return {
	[1] = {
		["type"] = 1,
		["resourceName"] = "Treadmill1",
		["id"] = 1,
		["iconImageId"] = "rbxassetid://119836096425123",
		["name_t"] = "Treadmill1",
		["costCoin"] = 0,
		["productId"] = 0,
		["costRb"] = 0,
		["ability"] = 1,
		["costType"] = 1,
		["quality"] = 1,
	},
	[2] = {
		["type"] = 1,
		["resourceName"] = "Treadmill2",
		["id"] = 2,
		["iconImageId"] = "rbxassetid://92022275554709",
		["name_t"] = "Treadmill2",
		["costCoin"] = 0,
		["productId"] = 0,
		["costRb"] = 0,
		["ability"] = 2,
		["costType"] = 4,
		["quality"] = 2,
	},
	[3] = {
		["type"] = 1,
		["resourceName"] = "Treadmill3",
		["id"] = 3,
		["iconImageId"] = "rbxassetid://129022654862676",
		["name_t"] = "Treadmill3",
		["costCoin"] = 3,
		["productId"] = 0,
		["costRb"] = 0,
		["ability"] = 3,
		["costType"] = 1,
		["quality"] = 3,
	},
	[4] = {
		["type"] = 2,
		["resourceName"] = "Bike1",
		["id"] = 4,
		["iconImageId"] = "rbxassetid://99426019150398",
		["name_t"] = "Bike1",
		["costCoin"] = 10,
		["productId"] = 3583247572,
		["costRb"] = 9,
		["ability"] = 4,
		["costType"] = 3,
		["quality"] = 4,
	},
	[5] = {
		["type"] = 2,
		["resourceName"] = "Bike2",
		["id"] = 5,
		["iconImageId"] = "rbxassetid://100504318605769",
		["name_t"] = "Bike2",
		["costCoin"] = 30,
		["productId"] = 3583247628,
		["costRb"] = 29,
		["ability"] = 5,
		["costType"] = 3,
		["quality"] = 5,
	},
	[6] = {
		["type"] = 2,
		["resourceName"] = "Bike3",
		["id"] = 6,
		["iconImageId"] = "rbxassetid://83654076503938",
		["name_t"] = "Bike3",
		["costCoin"] = 100,
		["productId"] = 3583247683,
		["costRb"] = 99,
		["ability"] = 6,
		["costType"] = 3,
		["quality"] = 6,
	},
}