return {
	[1] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill1",
		["id"] = 1,
		["iconImageId"] = "rbxassetid://119836096425123",
		["name_t"] = "Treadmill1",
		["Column14"] = 0,
		["costCoin"] = 100,
		["productId"] = 0,
		["costRb"] = 0,
		["ability"] = 1,
		["costType"] = 1,
		["quality"] = 1,
	},
	[2] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike1",
		["id"] = 2,
		["iconImageId"] = "rbxassetid://99426019150398",
		["name_t"] = "Bike1",
		["Column14"] = 0,
		["costCoin"] = 1500,
		["productId"] = 3583247572,
		["costRb"] = 9,
		["ability"] = 2.5,
		["costType"] = 3,
		["quality"] = 1,
	},
	[3] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill2",
		["id"] = 3,
		["iconImageId"] = "rbxassetid://92022275554709",
		["name_t"] = "Treadmill2",
		["Column14"] = 0,
		["costCoin"] = 23000,
		["productId"] = 0,
		["costRb"] = 25,
		["ability"] = 6.25,
		["costType"] = 3,
		["quality"] = 2,
	},
	[4] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike2",
		["id"] = 4,
		["iconImageId"] = "rbxassetid://100504318605769",
		["name_t"] = "Bike2",
		["Column14"] = 0,
		["costCoin"] = 340000,
		["productId"] = 3583247628,
		["costRb"] = 80,
		["ability"] = 15.6,
		["costType"] = 3,
		["quality"] = 2,
	},
	[5] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill3",
		["id"] = 5,
		["iconImageId"] = "rbxassetid://129022654862676",
		["name_t"] = "Treadmill3",
		["Column14"] = 0,
		["costCoin"] = 5100000,
		["productId"] = 0,
		["costRb"] = 160,
		["ability"] = 39.1,
		["costType"] = 3,
		["quality"] = 3,
	},
	[6] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike3",
		["id"] = 6,
		["iconImageId"] = "rbxassetid://83654076503938",
		["name_t"] = "Bike3",
		["Column14"] = 0,
		["costCoin"] = 76000000,
		["productId"] = 3583247683,
		["costRb"] = 280,
		["ability"] = 97.7,
		["costType"] = 3,
		["quality"] = 3,
	},
	[7] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill4",
		["id"] = 7,
		["iconImageId"] = "rbxassetid://119836096425123",
		["name_t"] = "Treadmill4",
		["Column14"] = 0,
		["costCoin"] = 1100000000,
		["productId"] = 0,
		["costRb"] = 400,
		["ability"] = 244,
		["costType"] = 3,
		["quality"] = 1,
	},
	[8] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike4",
		["id"] = 8,
		["iconImageId"] = "rbxassetid://99426019150398",
		["name_t"] = "Bike4",
		["Column14"] = 0,
		["costCoin"] = 17000000000,
		["productId"] = 0,
		["costRb"] = 480,
		["ability"] = 610,
		["costType"] = 3,
		["quality"] = 1,
	},
	[9] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill5",
		["id"] = 9,
		["iconImageId"] = "rbxassetid://92022275554709",
		["name_t"] = "Treadmill5",
		["Column14"] = 0,
		["costCoin"] = 260000000000,
		["productId"] = 0,
		["costRb"] = 636,
		["ability"] = 1530,
		["costType"] = 3,
		["quality"] = 2,
	},
	[10] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike5",
		["id"] = 10,
		["iconImageId"] = "rbxassetid://100504318605769",
		["name_t"] = "Bike5",
		["Column14"] = 0,
		["costCoin"] = 3800000000000,
		["productId"] = 0,
		["costRb"] = 720,
		["ability"] = 3810,
		["costType"] = 3,
		["quality"] = 2,
	},
	[11] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill6",
		["id"] = 11,
		["iconImageId"] = "rbxassetid://129022654862676",
		["name_t"] = "Treadmill6",
		["Column14"] = 0,
		["costCoin"] = 58000000000000,
		["productId"] = 0,
		["costRb"] = 960,
		["ability"] = 9540,
		["costType"] = 3,
		["quality"] = 3,
	},
	[12] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike6",
		["id"] = 12,
		["iconImageId"] = "rbxassetid://83654076503938",
		["name_t"] = "Bike6",
		["Column14"] = 0,
		["costCoin"] = 860000000000000,
		["productId"] = 0,
		["costRb"] = 1160,
		["ability"] = 23800,
		["costType"] = 3,
		["quality"] = 3,
	},
	[13] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill7",
		["id"] = 13,
		["iconImageId"] = "rbxassetid://119836096425123",
		["name_t"] = "Treadmill7",
		["Column14"] = 0,
		["costCoin"] = 13000000000000000,
		["productId"] = 0,
		["costRb"] = 1600,
		["ability"] = 59600,
		["costType"] = 3,
		["quality"] = 1,
	},
	[14] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike7",
		["id"] = 14,
		["iconImageId"] = "rbxassetid://83654076503938",
		["name_t"] = "Bike7",
		["Column14"] = 0,
		["costCoin"] = 190000000000000000,
		["productId"] = 0,
		["costRb"] = 1160,
		["ability"] = 149000,
		["costType"] = 3,
		["quality"] = 3,
	},
	[15] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill8",
		["id"] = 15,
		["iconImageId"] = "rbxassetid://119836096425123",
		["name_t"] = "Treadmill8",
		["Column14"] = 0,
		["costCoin"] = 2900000000000000000,
		["productId"] = 0,
		["costRb"] = 1600,
		["ability"] = 373000,
		["costType"] = 3,
		["quality"] = 1,
	},
}