return {
	[1] = {
		["type"] = 1,
		["exp"] = 10,
		["ability"] = 20,
		["lv"] = 1,
		["name_t"] = "Jumppower",
	},
	[2] = {
		["type"] = 1,
		["exp"] = 12,
		["ability"] = 22,
		["lv"] = 2,
		["name_t"] = "Jumppower",
	},
	[3] = {
		["type"] = 1,
		["exp"] = 14,
		["ability"] = 24,
		["lv"] = 3,
		["name_t"] = "Jumppower",
	},
	[4] = {
		["type"] = 1,
		["exp"] = 17,
		["ability"] = 26,
		["lv"] = 4,
		["name_t"] = "Jumppower",
	},
	[5] = {
		["type"] = 1,
		["exp"] = 21,
		["ability"] = 28,
		["lv"] = 5,
		["name_t"] = "Jumppower",
	},
	[6] = {
		["type"] = 1,
		["exp"] = 25,
		["ability"] = 30,
		["lv"] = 6,
		["name_t"] = "Jumppower",
	},
	[7] = {
		["type"] = 1,
		["exp"] = 30,
		["ability"] = 32,
		["lv"] = 7,
		["name_t"] = "Jumppower",
	},
	[8] = {
		["type"] = 1,
		["exp"] = 36,
		["ability"] = 34,
		["lv"] = 8,
		["name_t"] = "Jumppower",
	},
	[9] = {
		["type"] = 1,
		["exp"] = 43,
		["ability"] = 36,
		["lv"] = 9,
		["name_t"] = "Jumppower",
	},
	[10] = {
		["type"] = 1,
		["exp"] = 52,
		["ability"] = 38,
		["lv"] = 10,
		["name_t"] = "Jumppower",
	},
	[11] = {
		["type"] = 1,
		["exp"] = 62,
		["ability"] = 40,
		["lv"] = 11,
		["name_t"] = "Jumppower",
	},
	[12] = {
		["type"] = 1,
		["exp"] = 74,
		["ability"] = 42,
		["lv"] = 12,
		["name_t"] = "Jumppower",
	},
	[13] = {
		["type"] = 1,
		["exp"] = 89,
		["ability"] = 44,
		["lv"] = 13,
		["name_t"] = "Jumppower",
	},
	[14] = {
		["type"] = 1,
		["exp"] = 110,
		["ability"] = 46,
		["lv"] = 14,
		["name_t"] = "Jumppower",
	},
	[15] = {
		["type"] = 1,
		["exp"] = 130,
		["ability"] = 48,
		["lv"] = 15,
		["name_t"] = "Jumppower",
	},
	[16] = {
		["type"] = 1,
		["exp"] = 150,
		["ability"] = 50,
		["lv"] = 16,
		["name_t"] = "Jumppower",
	},
	[17] = {
		["type"] = 1,
		["exp"] = 180,
		["ability"] = 52,
		["lv"] = 17,
		["name_t"] = "Jumppower",
	},
	[18] = {
		["type"] = 1,
		["exp"] = 220,
		["ability"] = 54,
		["lv"] = 18,
		["name_t"] = "Jumppower",
	},
	[19] = {
		["type"] = 1,
		["exp"] = 270,
		["ability"] = 56,
		["lv"] = 19,
		["name_t"] = "Jumppower",
	},
	[20] = {
		["type"] = 1,
		["exp"] = 320,
		["ability"] = 58,
		["lv"] = 20,
		["name_t"] = "Jumppower",
	},
	[21] = {
		["type"] = 1,
		["exp"] = 380,
		["ability"] = 60,
		["lv"] = 21,
		["name_t"] = "Jumppower",
	},
	[22] = {
		["type"] = 1,
		["exp"] = 460,
		["ability"] = 62,
		["lv"] = 22,
		["name_t"] = "Jumppower",
	},
	[23] = {
		["type"] = 1,
		["exp"] = 550,
		["ability"] = 64,
		["lv"] = 23,
		["name_t"] = "Jumppower",
	},
	[24] = {
		["type"] = 1,
		["exp"] = 660,
		["ability"] = 66,
		["lv"] = 24,
		["name_t"] = "Jumppower",
	},
	[25] = {
		["type"] = 1,
		["exp"] = 790,
		["ability"] = 68,
		["lv"] = 25,
		["name_t"] = "Jumppower",
	},
	[26] = {
		["type"] = 1,
		["exp"] = 950,
		["ability"] = 70,
		["lv"] = 26,
		["name_t"] = "Jumppower",
	},
	[27] = {
		["type"] = 1,
		["exp"] = 1100,
		["ability"] = 72,
		["lv"] = 27,
		["name_t"] = "Jumppower",
	},
	[28] = {
		["type"] = 1,
		["exp"] = 1400,
		["ability"] = 74,
		["lv"] = 28,
		["name_t"] = "Jumppower",
	},
	[29] = {
		["type"] = 1,
		["exp"] = 1600,
		["ability"] = 76,
		["lv"] = 29,
		["name_t"] = "Jumppower",
	},
	[30] = {
		["type"] = 1,
		["exp"] = 2000,
		["ability"] = 78,
		["lv"] = 30,
		["name_t"] = "Jumppower",
	},
	[31] = {
		["type"] = 1,
		["exp"] = 2400,
		["ability"] = 80,
		["lv"] = 31,
		["name_t"] = "Jumppower",
	},
	[32] = {
		["type"] = 1,
		["exp"] = 2800,
		["ability"] = 82,
		["lv"] = 32,
		["name_t"] = "Jumppower",
	},
	[33] = {
		["type"] = 1,
		["exp"] = 3400,
		["ability"] = 84,
		["lv"] = 33,
		["name_t"] = "Jumppower",
	},
	[34] = {
		["type"] = 1,
		["exp"] = 4100,
		["ability"] = 86,
		["lv"] = 34,
		["name_t"] = "Jumppower",
	},
	[35] = {
		["type"] = 1,
		["exp"] = 4900,
		["ability"] = 88,
		["lv"] = 35,
		["name_t"] = "Jumppower",
	},
	[36] = {
		["type"] = 1,
		["exp"] = 5900,
		["ability"] = 90,
		["lv"] = 36,
		["name_t"] = "Jumppower",
	},
	[37] = {
		["type"] = 1,
		["exp"] = 7100,
		["ability"] = 92,
		["lv"] = 37,
		["name_t"] = "Jumppower",
	},
	[38] = {
		["type"] = 1,
		["exp"] = 8500,
		["ability"] = 94,
		["lv"] = 38,
		["name_t"] = "Jumppower",
	},
	[39] = {
		["type"] = 1,
		["exp"] = 10000,
		["ability"] = 96,
		["lv"] = 39,
		["name_t"] = "Jumppower",
	},
	[40] = {
		["type"] = 1,
		["exp"] = 12000,
		["ability"] = 98,
		["lv"] = 40,
		["name_t"] = "Jumppower",
	},
	[41] = {
		["type"] = 1,
		["exp"] = 15000,
		["ability"] = 100,
		["lv"] = 41,
		["name_t"] = "Jumppower",
	},
	[42] = {
		["type"] = 1,
		["exp"] = 18000,
		["ability"] = 102,
		["lv"] = 42,
		["name_t"] = "Jumppower",
	},
	[43] = {
		["type"] = 1,
		["exp"] = 21000,
		["ability"] = 104,
		["lv"] = 43,
		["name_t"] = "Jumppower",
	},
	[44] = {
		["type"] = 1,
		["exp"] = 25000,
		["ability"] = 106,
		["lv"] = 44,
		["name_t"] = "Jumppower",
	},
	[45] = {
		["type"] = 1,
		["exp"] = 30000,
		["ability"] = 108,
		["lv"] = 45,
		["name_t"] = "Jumppower",
	},
	[46] = {
		["type"] = 1,
		["exp"] = 37000,
		["ability"] = 110,
		["lv"] = 46,
		["name_t"] = "Jumppower",
	},
	[47] = {
		["type"] = 1,
		["exp"] = 44000,
		["ability"] = 112,
		["lv"] = 47,
		["name_t"] = "Jumppower",
	},
	[48] = {
		["type"] = 1,
		["exp"] = 53000,
		["ability"] = 114,
		["lv"] = 48,
		["name_t"] = "Jumppower",
	},
	[49] = {
		["type"] = 1,
		["exp"] = 63000,
		["ability"] = 116,
		["lv"] = 49,
		["name_t"] = "Jumppower",
	},
	[50] = {
		["type"] = 1,
		["exp"] = 76000,
		["ability"] = 118,
		["lv"] = 50,
		["name_t"] = "Jumppower",
	},
	[51] = {
		["type"] = 1,
		["exp"] = 91000,
		["ability"] = 120,
		["lv"] = 51,
		["name_t"] = "Jumppower",
	},
	[52] = {
		["type"] = 1,
		["exp"] = 110000,
		["ability"] = 122,
		["lv"] = 52,
		["name_t"] = "Jumppower",
	},
	[53] = {
		["type"] = 1,
		["exp"] = 130000,
		["ability"] = 124,
		["lv"] = 53,
		["name_t"] = "Jumppower",
	},
	[54] = {
		["type"] = 1,
		["exp"] = 160000,
		["ability"] = 126,
		["lv"] = 54,
		["name_t"] = "Jumppower",
	},
	[55] = {
		["type"] = 1,
		["exp"] = 190000,
		["ability"] = 128,
		["lv"] = 55,
		["name_t"] = "Jumppower",
	},
	[56] = {
		["type"] = 1,
		["exp"] = 230000,
		["ability"] = 130,
		["lv"] = 56,
		["name_t"] = "Jumppower",
	},
	[57] = {
		["type"] = 1,
		["exp"] = 270000,
		["ability"] = 132,
		["lv"] = 57,
		["name_t"] = "Jumppower",
	},
	[58] = {
		["type"] = 1,
		["exp"] = 330000,
		["ability"] = 134,
		["lv"] = 58,
		["name_t"] = "Jumppower",
	},
	[59] = {
		["type"] = 1,
		["exp"] = 390000,
		["ability"] = 136,
		["lv"] = 59,
		["name_t"] = "Jumppower",
	},
	[60] = {
		["type"] = 1,
		["exp"] = 470000,
		["ability"] = 138,
		["lv"] = 60,
		["name_t"] = "Jumppower",
	},
	[61] = {
		["type"] = 1,
		["exp"] = 560000,
		["ability"] = 140,
		["lv"] = 61,
		["name_t"] = "Jumppower",
	},
	[62] = {
		["type"] = 1,
		["exp"] = 680000,
		["ability"] = 142,
		["lv"] = 62,
		["name_t"] = "Jumppower",
	},
	[63] = {
		["type"] = 1,
		["exp"] = 810000,
		["ability"] = 144,
		["lv"] = 63,
		["name_t"] = "Jumppower",
	},
	[64] = {
		["type"] = 1,
		["exp"] = 970000,
		["ability"] = 146,
		["lv"] = 64,
		["name_t"] = "Jumppower",
	},
	[65] = {
		["type"] = 1,
		["exp"] = 1200000,
		["ability"] = 148,
		["lv"] = 65,
		["name_t"] = "Jumppower",
	},
	[66] = {
		["type"] = 1,
		["exp"] = 1400000,
		["ability"] = 150,
		["lv"] = 66,
		["name_t"] = "Jumppower",
	},
	[67] = {
		["type"] = 1,
		["exp"] = 1700000,
		["ability"] = 152,
		["lv"] = 67,
		["name_t"] = "Jumppower",
	},
	[68] = {
		["type"] = 1,
		["exp"] = 2000000,
		["ability"] = 154,
		["lv"] = 68,
		["name_t"] = "Jumppower",
	},
	[69] = {
		["type"] = 1,
		["exp"] = 2400000,
		["ability"] = 156,
		["lv"] = 69,
		["name_t"] = "Jumppower",
	},
	[70] = {
		["type"] = 1,
		["exp"] = 2900000,
		["ability"] = 158,
		["lv"] = 70,
		["name_t"] = "Jumppower",
	},
	[71] = {
		["type"] = 1,
		["exp"] = 3500000,
		["ability"] = 160,
		["lv"] = 71,
		["name_t"] = "Jumppower",
	},
	[72] = {
		["type"] = 1,
		["exp"] = 4200000,
		["ability"] = 162,
		["lv"] = 72,
		["name_t"] = "Jumppower",
	},
	[73] = {
		["type"] = 1,
		["exp"] = 5000000,
		["ability"] = 164,
		["lv"] = 73,
		["name_t"] = "Jumppower",
	},
	[74] = {
		["type"] = 1,
		["exp"] = 6000000,
		["ability"] = 166,
		["lv"] = 74,
		["name_t"] = "Jumppower",
	},
	[75] = {
		["type"] = 1,
		["exp"] = 7200000,
		["ability"] = 168,
		["lv"] = 75,
		["name_t"] = "Jumppower",
	},
	[76] = {
		["type"] = 1,
		["exp"] = 8700000,
		["ability"] = 170,
		["lv"] = 76,
		["name_t"] = "Jumppower",
	},
	[77] = {
		["type"] = 1,
		["exp"] = 10000000,
		["ability"] = 172,
		["lv"] = 77,
		["name_t"] = "Jumppower",
	},
	[78] = {
		["type"] = 1,
		["exp"] = 13000000,
		["ability"] = 174,
		["lv"] = 78,
		["name_t"] = "Jumppower",
	},
	[79] = {
		["type"] = 1,
		["exp"] = 15000000,
		["ability"] = 176,
		["lv"] = 79,
		["name_t"] = "Jumppower",
	},
	[80] = {
		["type"] = 1,
		["exp"] = 18000000,
		["ability"] = 178,
		["lv"] = 80,
		["name_t"] = "Jumppower",
	},
	[81] = {
		["type"] = 1,
		["exp"] = 22000000,
		["ability"] = 180,
		["lv"] = 81,
		["name_t"] = "Jumppower",
	},
	[82] = {
		["type"] = 1,
		["exp"] = 26000000,
		["ability"] = 182,
		["lv"] = 82,
		["name_t"] = "Jumppower",
	},
	[83] = {
		["type"] = 1,
		["exp"] = 31000000,
		["ability"] = 184,
		["lv"] = 83,
		["name_t"] = "Jumppower",
	},
	[84] = {
		["type"] = 1,
		["exp"] = 37000000,
		["ability"] = 186,
		["lv"] = 84,
		["name_t"] = "Jumppower",
	},
	[85] = {
		["type"] = 1,
		["exp"] = 45000000,
		["ability"] = 188,
		["lv"] = 85,
		["name_t"] = "Jumppower",
	},
	[86] = {
		["type"] = 1,
		["exp"] = 54000000,
		["ability"] = 190,
		["lv"] = 86,
		["name_t"] = "Jumppower",
	},
	[87] = {
		["type"] = 1,
		["exp"] = 65000000,
		["ability"] = 192,
		["lv"] = 87,
		["name_t"] = "Jumppower",
	},
	[88] = {
		["type"] = 1,
		["exp"] = 77000000,
		["ability"] = 194,
		["lv"] = 88,
		["name_t"] = "Jumppower",
	},
	[89] = {
		["type"] = 1,
		["exp"] = 93000000,
		["ability"] = 196,
		["lv"] = 89,
		["name_t"] = "Jumppower",
	},
	[90] = {
		["type"] = 1,
		["exp"] = 110000000,
		["ability"] = 198,
		["lv"] = 90,
		["name_t"] = "Jumppower",
	},
	[91] = {
		["type"] = 1,
		["exp"] = 130000000,
		["ability"] = 200,
		["lv"] = 91,
		["name_t"] = "Jumppower",
	},
	[92] = {
		["type"] = 1,
		["exp"] = 160000000,
		["ability"] = 202,
		["lv"] = 92,
		["name_t"] = "Jumppower",
	},
	[93] = {
		["type"] = 1,
		["exp"] = 190000000,
		["ability"] = 204,
		["lv"] = 93,
		["name_t"] = "Jumppower",
	},
	[94] = {
		["type"] = 1,
		["exp"] = 230000000,
		["ability"] = 206,
		["lv"] = 94,
		["name_t"] = "Jumppower",
	},
	[95] = {
		["type"] = 1,
		["exp"] = 280000000,
		["ability"] = 208,
		["lv"] = 95,
		["name_t"] = "Jumppower",
	},
	[96] = {
		["type"] = 1,
		["exp"] = 330000000,
		["ability"] = 210,
		["lv"] = 96,
		["name_t"] = "Jumppower",
	},
	[97] = {
		["type"] = 1,
		["exp"] = 400000000,
		["ability"] = 212,
		["lv"] = 97,
		["name_t"] = "Jumppower",
	},
	[98] = {
		["type"] = 1,
		["exp"] = 480000000,
		["ability"] = 214,
		["lv"] = 98,
		["name_t"] = "Jumppower",
	},
	[99] = {
		["type"] = 1,
		["exp"] = 580000000,
		["ability"] = 216,
		["lv"] = 99,
		["name_t"] = "Jumppower",
	},
	[100] = {
		["type"] = 1,
		["exp"] = 690000000,
		["ability"] = 218,
		["lv"] = 100,
		["name_t"] = "Jumppower",
	},
	[101] = {
		["type"] = 1,
		["exp"] = 830000000,
		["ability"] = 220,
		["lv"] = 101,
		["name_t"] = "Jumppower",
	},
	[102] = {
		["type"] = 1,
		["exp"] = 990000000,
		["ability"] = 222,
		["lv"] = 102,
		["name_t"] = "Jumppower",
	},
	[103] = {
		["type"] = 1,
		["exp"] = 1200000000,
		["ability"] = 224,
		["lv"] = 103,
		["name_t"] = "Jumppower",
	},
	[104] = {
		["type"] = 1,
		["exp"] = 1400000000,
		["ability"] = 226,
		["lv"] = 104,
		["name_t"] = "Jumppower",
	},
	[105] = {
		["type"] = 1,
		["exp"] = 1700000000,
		["ability"] = 228,
		["lv"] = 105,
		["name_t"] = "Jumppower",
	},
	[106] = {
		["type"] = 1,
		["exp"] = 2100000000,
		["ability"] = 230,
		["lv"] = 106,
		["name_t"] = "Jumppower",
	},
	[107] = {
		["type"] = 1,
		["exp"] = 2500000000,
		["ability"] = 232,
		["lv"] = 107,
		["name_t"] = "Jumppower",
	},
	[108] = {
		["type"] = 1,
		["exp"] = 3000000000,
		["ability"] = 234,
		["lv"] = 108,
		["name_t"] = "Jumppower",
	},
	[109] = {
		["type"] = 1,
		["exp"] = 3600000000,
		["ability"] = 236,
		["lv"] = 109,
		["name_t"] = "Jumppower",
	},
	[110] = {
		["type"] = 1,
		["exp"] = 4300000000,
		["ability"] = 238,
		["lv"] = 110,
		["name_t"] = "Jumppower",
	},
	[111] = {
		["type"] = 1,
		["exp"] = 5100000000,
		["ability"] = 240,
		["lv"] = 111,
		["name_t"] = "Jumppower",
	},
	[112] = {
		["type"] = 1,
		["exp"] = 6200000000,
		["ability"] = 242,
		["lv"] = 112,
		["name_t"] = "Jumppower",
	},
	[113] = {
		["type"] = 1,
		["exp"] = 7400000000,
		["ability"] = 244,
		["lv"] = 113,
		["name_t"] = "Jumppower",
	},
	[114] = {
		["type"] = 1,
		["exp"] = 8900000000,
		["ability"] = 246,
		["lv"] = 114,
		["name_t"] = "Jumppower",
	},
	[115] = {
		["type"] = 1,
		["exp"] = 11000000000,
		["ability"] = 248,
		["lv"] = 115,
		["name_t"] = "Jumppower",
	},
	[116] = {
		["type"] = 1,
		["exp"] = 13000000000,
		["ability"] = 250,
		["lv"] = 116,
		["name_t"] = "Jumppower",
	},
	[117] = {
		["type"] = 1,
		["exp"] = 15000000000,
		["ability"] = 252,
		["lv"] = 117,
		["name_t"] = "Jumppower",
	},
	[118] = {
		["type"] = 1,
		["exp"] = 18000000000,
		["ability"] = 254,
		["lv"] = 118,
		["name_t"] = "Jumppower",
	},
	[119] = {
		["type"] = 1,
		["exp"] = 22000000000,
		["ability"] = 256,
		["lv"] = 119,
		["name_t"] = "Jumppower",
	},
	[120] = {
		["type"] = 1,
		["exp"] = 26000000000,
		["ability"] = 258,
		["lv"] = 120,
		["name_t"] = "Jumppower",
	},
	[121] = {
		["type"] = 1,
		["exp"] = 32000000000,
		["ability"] = 260,
		["lv"] = 121,
		["name_t"] = "Jumppower",
	},
	[122] = {
		["type"] = 1,
		["exp"] = 38000000000,
		["ability"] = 262,
		["lv"] = 122,
		["name_t"] = "Jumppower",
	},
	[123] = {
		["type"] = 1,
		["exp"] = 46000000000,
		["ability"] = 264,
		["lv"] = 123,
		["name_t"] = "Jumppower",
	},
	[124] = {
		["type"] = 1,
		["exp"] = 55000000000,
		["ability"] = 266,
		["lv"] = 124,
		["name_t"] = "Jumppower",
	},
	[125] = {
		["type"] = 1,
		["exp"] = 66000000000,
		["ability"] = 268,
		["lv"] = 125,
		["name_t"] = "Jumppower",
	},
	[126] = {
		["type"] = 1,
		["exp"] = 79000000000,
		["ability"] = 270,
		["lv"] = 126,
		["name_t"] = "Jumppower",
	},
	[127] = {
		["type"] = 1,
		["exp"] = 95000000000,
		["ability"] = 272,
		["lv"] = 127,
		["name_t"] = "Jumppower",
	},
	[128] = {
		["type"] = 1,
		["exp"] = 110000000000,
		["ability"] = 274,
		["lv"] = 128,
		["name_t"] = "Jumppower",
	},
	[129] = {
		["type"] = 1,
		["exp"] = 140000000000,
		["ability"] = 276,
		["lv"] = 129,
		["name_t"] = "Jumppower",
	},
	[130] = {
		["type"] = 1,
		["exp"] = 160000000000,
		["ability"] = 278,
		["lv"] = 130,
		["name_t"] = "Jumppower",
	},
	[131] = {
		["type"] = 1,
		["exp"] = 200000000000,
		["ability"] = 280,
		["lv"] = 131,
		["name_t"] = "Jumppower",
	},
	[132] = {
		["type"] = 0,
		["exp"] = 240000000000,
		["ability"] = 282,
		["lv"] = 132,
		["name_t"] = "Jumppower",
	},
	[133] = {
		["type"] = 0,
		["exp"] = 280000000000,
		["ability"] = 284,
		["lv"] = 133,
		["name_t"] = "Jumppower",
	},
	[134] = {
		["type"] = 0,
		["exp"] = 340000000000,
		["ability"] = 286,
		["lv"] = 134,
		["name_t"] = "Jumppower",
	},
	[135] = {
		["type"] = 0,
		["exp"] = 410000000000,
		["ability"] = 288,
		["lv"] = 135,
		["name_t"] = "Jumppower",
	},
	[136] = {
		["type"] = 0,
		["exp"] = 490000000000,
		["ability"] = 290,
		["lv"] = 136,
		["name_t"] = "Jumppower",
	},
	[137] = {
		["type"] = 0,
		["exp"] = 590000000000,
		["ability"] = 292,
		["lv"] = 137,
		["name_t"] = "Jumppower",
	},
	[138] = {
		["type"] = 0,
		["exp"] = 700000000000,
		["ability"] = 294,
		["lv"] = 138,
		["name_t"] = "Jumppower",
	},
	[139] = {
		["type"] = 0,
		["exp"] = 850000000000,
		["ability"] = 296,
		["lv"] = 139,
		["name_t"] = "Jumppower",
	},
	[140] = {
		["type"] = 0,
		["exp"] = 1000000000000,
		["ability"] = 298,
		["lv"] = 140,
		["name_t"] = "Jumppower",
	},
	[141] = {
		["type"] = 0,
		["exp"] = 1200000000000,
		["ability"] = 300,
		["lv"] = 141,
		["name_t"] = "Jumppower",
	},
	[142] = {
		["type"] = 0,
		["exp"] = 1500000000000,
		["ability"] = 302,
		["lv"] = 142,
		["name_t"] = "Jumppower",
	},
	[143] = {
		["type"] = 0,
		["exp"] = 1800000000000,
		["ability"] = 304,
		["lv"] = 143,
		["name_t"] = "Jumppower",
	},
	[144] = {
		["type"] = 0,
		["exp"] = 2100000000000,
		["ability"] = 306,
		["lv"] = 144,
		["name_t"] = "Jumppower",
	},
	[145] = {
		["type"] = 0,
		["exp"] = 2500000000000,
		["ability"] = 308,
		["lv"] = 145,
		["name_t"] = "Jumppower",
	},
	[146] = {
		["type"] = 0,
		["exp"] = 3000000000000,
		["ability"] = 310,
		["lv"] = 146,
		["name_t"] = "Jumppower",
	},
	[147] = {
		["type"] = 0,
		["exp"] = 3600000000000,
		["ability"] = 312,
		["lv"] = 147,
		["name_t"] = "Jumppower",
	},
	[148] = {
		["type"] = 0,
		["exp"] = 4400000000000,
		["ability"] = 314,
		["lv"] = 148,
		["name_t"] = "Jumppower",
	},
	[149] = {
		["type"] = 0,
		["exp"] = 5200000000000,
		["ability"] = 316,
		["lv"] = 149,
		["name_t"] = "Jumppower",
	},
	[150] = {
		["type"] = 0,
		["exp"] = 6300000000000,
		["ability"] = 318,
		["lv"] = 150,
		["name_t"] = "Jumppower",
	},
	[151] = {
		["type"] = 0,
		["exp"] = 7500000000000,
		["ability"] = 320,
		["lv"] = 151,
		["name_t"] = "Jumppower",
	},
	[152] = {
		["type"] = 0,
		["exp"] = 9000000000000,
		["ability"] = 322,
		["lv"] = 152,
		["name_t"] = "Jumppower",
	},
	[153] = {
		["type"] = 0,
		["exp"] = 11000000000000,
		["ability"] = 324,
		["lv"] = 153,
		["name_t"] = "Jumppower",
	},
	[154] = {
		["type"] = 0,
		["exp"] = 13000000000000,
		["ability"] = 326,
		["lv"] = 154,
		["name_t"] = "Jumppower",
	},
	[155] = {
		["type"] = 0,
		["exp"] = 16000000000000,
		["ability"] = 328,
		["lv"] = 155,
		["name_t"] = "Jumppower",
	},
	[156] = {
		["type"] = 0,
		["exp"] = 19000000000000,
		["ability"] = 330,
		["lv"] = 156,
		["name_t"] = "Jumppower",
	},
	[157] = {
		["type"] = 0,
		["exp"] = 23000000000000,
		["ability"] = 332,
		["lv"] = 157,
		["name_t"] = "Jumppower",
	},
	[158] = {
		["type"] = 0,
		["exp"] = 27000000000000,
		["ability"] = 334,
		["lv"] = 158,
		["name_t"] = "Jumppower",
	},
	[159] = {
		["type"] = 0,
		["exp"] = 32000000000000,
		["ability"] = 336,
		["lv"] = 159,
		["name_t"] = "Jumppower",
	},
	[160] = {
		["type"] = 0,
		["exp"] = 39000000000000,
		["ability"] = 338,
		["lv"] = 160,
		["name_t"] = "Jumppower",
	},
	[161] = {
		["type"] = 0,
		["exp"] = 47000000000000,
		["ability"] = 340,
		["lv"] = 161,
		["name_t"] = "Jumppower",
	},
	[162] = {
		["type"] = 0,
		["exp"] = 56000000000000,
		["ability"] = 342,
		["lv"] = 162,
		["name_t"] = "Jumppower",
	},
	[163] = {
		["type"] = 0,
		["exp"] = 67000000000000,
		["ability"] = 344,
		["lv"] = 163,
		["name_t"] = "Jumppower",
	},
	[164] = {
		["type"] = 0,
		["exp"] = 81000000000000,
		["ability"] = 346,
		["lv"] = 164,
		["name_t"] = "Jumppower",
	},
	[165] = {
		["type"] = 0,
		["exp"] = 97000000000000,
		["ability"] = 348,
		["lv"] = 165,
		["name_t"] = "Jumppower",
	},
	[166] = {
		["type"] = 0,
		["exp"] = 120000000000000,
		["ability"] = 350,
		["lv"] = 166,
		["name_t"] = "Jumppower",
	},
	[167] = {
		["type"] = 0,
		["exp"] = 140000000000000,
		["ability"] = 352,
		["lv"] = 167,
		["name_t"] = "Jumppower",
	},
	[168] = {
		["type"] = 0,
		["exp"] = 170000000000000,
		["ability"] = 354,
		["lv"] = 168,
		["name_t"] = "Jumppower",
	},
	[169] = {
		["type"] = 0,
		["exp"] = 200000000000000,
		["ability"] = 356,
		["lv"] = 169,
		["name_t"] = "Jumppower",
	},
	[170] = {
		["type"] = 0,
		["exp"] = 240000000000000,
		["ability"] = 358,
		["lv"] = 170,
		["name_t"] = "Jumppower",
	},
	[171] = {
		["type"] = 0,
		["exp"] = 290000000000000,
		["ability"] = 360,
		["lv"] = 171,
		["name_t"] = "Jumppower",
	},
	[172] = {
		["type"] = 0,
		["exp"] = 350000000000000,
		["ability"] = 362,
		["lv"] = 172,
		["name_t"] = "Jumppower",
	},
	[173] = {
		["type"] = 0,
		["exp"] = 420000000000000,
		["ability"] = 364,
		["lv"] = 173,
		["name_t"] = "Jumppower",
	},
	[174] = {
		["type"] = 0,
		["exp"] = 500000000000000,
		["ability"] = 366,
		["lv"] = 174,
		["name_t"] = "Jumppower",
	},
	[175] = {
		["type"] = 0,
		["exp"] = 600000000000000,
		["ability"] = 368,
		["lv"] = 175,
		["name_t"] = "Jumppower",
	},
	[176] = {
		["type"] = 0,
		["exp"] = 720000000000000,
		["ability"] = 370,
		["lv"] = 176,
		["name_t"] = "Jumppower",
	},
	[177] = {
		["type"] = 0,
		["exp"] = 860000000000000,
		["ability"] = 372,
		["lv"] = 177,
		["name_t"] = "Jumppower",
	},
	[178] = {
		["type"] = 0,
		["exp"] = 1000000000000000,
		["ability"] = 374,
		["lv"] = 178,
		["name_t"] = "Jumppower",
	},
	[179] = {
		["type"] = 0,
		["exp"] = 1200000000000000,
		["ability"] = 376,
		["lv"] = 179,
		["name_t"] = "Jumppower",
	},
	[180] = {
		["type"] = 0,
		["exp"] = 1500000000000000,
		["ability"] = 378,
		["lv"] = 180,
		["name_t"] = "Jumppower",
	},
	[181] = {
		["type"] = 0,
		["exp"] = 1800000000000000,
		["ability"] = 380,
		["lv"] = 181,
		["name_t"] = "Jumppower",
	},
	[182] = {
		["type"] = 0,
		["exp"] = 2100000000000000,
		["ability"] = 382,
		["lv"] = 182,
		["name_t"] = "Jumppower",
	},
	[183] = {
		["type"] = 0,
		["exp"] = 2600000000000000,
		["ability"] = 384,
		["lv"] = 183,
		["name_t"] = "Jumppower",
	},
	[184] = {
		["type"] = 0,
		["exp"] = 3100000000000000,
		["ability"] = 386,
		["lv"] = 184,
		["name_t"] = "Jumppower",
	},
	[185] = {
		["type"] = 0,
		["exp"] = 3700000000000000,
		["ability"] = 388,
		["lv"] = 185,
		["name_t"] = "Jumppower",
	},
	[186] = {
		["type"] = 0,
		["exp"] = 4500000000000000,
		["ability"] = 390,
		["lv"] = 186,
		["name_t"] = "Jumppower",
	},
	[187] = {
		["type"] = 0,
		["exp"] = 5300000000000000,
		["ability"] = 392,
		["lv"] = 187,
		["name_t"] = "Jumppower",
	},
	[188] = {
		["type"] = 0,
		["exp"] = 6400000000000000,
		["ability"] = 394,
		["lv"] = 188,
		["name_t"] = "Jumppower",
	},
	[189] = {
		["type"] = 0,
		["exp"] = 7700000000000000,
		["ability"] = 396,
		["lv"] = 189,
		["name_t"] = "Jumppower",
	},
	[190] = {
		["type"] = 0,
		["exp"] = 9200000000000000,
		["ability"] = 398,
		["lv"] = 190,
		["name_t"] = "Jumppower",
	},
	[191] = {
		["type"] = 0,
		["exp"] = 11000000000000000,
		["ability"] = 400,
		["lv"] = 191,
		["name_t"] = "Jumppower",
	},
	[192] = {
		["type"] = 0,
		["exp"] = 13000000000000000,
		["ability"] = 402,
		["lv"] = 192,
		["name_t"] = "Jumppower",
	},
	[193] = {
		["type"] = 0,
		["exp"] = 16000000000000000,
		["ability"] = 404,
		["lv"] = 193,
		["name_t"] = "Jumppower",
	},
	[194] = {
		["type"] = 0,
		["exp"] = 19000000000000000,
		["ability"] = 406,
		["lv"] = 194,
		["name_t"] = "Jumppower",
	},
	[195] = {
		["type"] = 0,
		["exp"] = 23000000000000000,
		["ability"] = 408,
		["lv"] = 195,
		["name_t"] = "Jumppower",
	},
	[196] = {
		["type"] = 0,
		["exp"] = 28000000000000000,
		["ability"] = 410,
		["lv"] = 196,
		["name_t"] = "Jumppower",
	},
	[197] = {
		["type"] = 0,
		["exp"] = 33000000000000000,
		["ability"] = 412,
		["lv"] = 197,
		["name_t"] = "Jumppower",
	},
	[198] = {
		["type"] = 0,
		["exp"] = 40000000000000000,
		["ability"] = 414,
		["lv"] = 198,
		["name_t"] = "Jumppower",
	},
	[199] = {
		["type"] = 0,
		["exp"] = 48000000000000000,
		["ability"] = 416,
		["lv"] = 199,
		["name_t"] = "Jumppower",
	},
	[200] = {
		["type"] = 0,
		["exp"] = 57000000000000000,
		["ability"] = 418,
		["lv"] = 200,
		["name_t"] = "Jumppower",
	},
	[201] = {
		["type"] = 0,
		["exp"] = 69000000000000000,
		["ability"] = 420,
		["lv"] = 201,
		["name_t"] = "Jumppower",
	},
	[202] = {
		["type"] = 0,
		["exp"] = 82000000000000000,
		["ability"] = 422,
		["lv"] = 202,
		["name_t"] = "Jumppower",
	},
	[203] = {
		["type"] = 0,
		["exp"] = 99000000000000000,
		["ability"] = 424,
		["lv"] = 203,
		["name_t"] = "Jumppower",
	},
	[204] = {
		["type"] = 0,
		["exp"] = 120000000000000000,
		["ability"] = 426,
		["lv"] = 204,
		["name_t"] = "Jumppower",
	},
	[205] = {
		["type"] = 0,
		["exp"] = 140000000000000000,
		["ability"] = 428,
		["lv"] = 205,
		["name_t"] = "Jumppower",
	},
	[206] = {
		["type"] = 0,
		["exp"] = 170000000000000000,
		["ability"] = 430,
		["lv"] = 206,
		["name_t"] = "Jumppower",
	},
	[207] = {
		["type"] = 0,
		["exp"] = 200000000000000000,
		["ability"] = 432,
		["lv"] = 207,
		["name_t"] = "Jumppower",
	},
	[208] = {
		["type"] = 0,
		["exp"] = 250000000000000000,
		["ability"] = 434,
		["lv"] = 208,
		["name_t"] = "Jumppower",
	},
	[209] = {
		["type"] = 0,
		["exp"] = 290000000000000000,
		["ability"] = 436,
		["lv"] = 209,
		["name_t"] = "Jumppower",
	},
	[210] = {
		["type"] = 0,
		["exp"] = 350000000000000000,
		["ability"] = 438,
		["lv"] = 210,
		["name_t"] = "Jumppower",
	},
	[211] = {
		["type"] = 0,
		["exp"] = 420000000000000000,
		["ability"] = 440,
		["lv"] = 211,
		["name_t"] = "Jumppower",
	},
	[212] = {
		["type"] = 0,
		["exp"] = 510000000000000000,
		["ability"] = 442,
		["lv"] = 212,
		["name_t"] = "Jumppower",
	},
	[213] = {
		["type"] = 0,
		["exp"] = 610000000000000000,
		["ability"] = 444,
		["lv"] = 213,
		["name_t"] = "Jumppower",
	},
	[214] = {
		["type"] = 0,
		["exp"] = 730000000000000000,
		["ability"] = 446,
		["lv"] = 214,
		["name_t"] = "Jumppower",
	},
	[215] = {
		["type"] = 0,
		["exp"] = 880000000000000000,
		["ability"] = 448,
		["lv"] = 215,
		["name_t"] = "Jumppower",
	},
	[216] = {
		["type"] = 0,
		["exp"] = 1100000000000000000,
		["ability"] = 450,
		["lv"] = 216,
		["name_t"] = "Jumppower",
	},
	[217] = {
		["type"] = 0,
		["exp"] = 1300000000000000000,
		["ability"] = 452,
		["lv"] = 217,
		["name_t"] = "Jumppower",
	},
	[218] = {
		["type"] = 0,
		["exp"] = 1500000000000000000,
		["ability"] = 454,
		["lv"] = 218,
		["name_t"] = "Jumppower",
	},
	[219] = {
		["type"] = 0,
		["exp"] = 1800000000000000000,
		["ability"] = 456,
		["lv"] = 219,
		["name_t"] = "Jumppower",
	},
	[220] = {
		["type"] = 0,
		["exp"] = 2200000000000000000,
		["ability"] = 458,
		["lv"] = 220,
		["name_t"] = "Jumppower",
	},
	[221] = {
		["type"] = 0,
		["exp"] = 2600000000000000000,
		["ability"] = 460,
		["lv"] = 221,
		["name_t"] = "Jumppower",
	},
	[222] = {
		["type"] = 0,
		["exp"] = 3200000000000000000,
		["ability"] = 462,
		["lv"] = 222,
		["name_t"] = "Jumppower",
	},
	[223] = {
		["type"] = 0,
		["exp"] = 3800000000000000000,
		["ability"] = 464,
		["lv"] = 223,
		["name_t"] = "Jumppower",
	},
	[224] = {
		["type"] = 0,
		["exp"] = 4500000000000000000,
		["ability"] = 466,
		["lv"] = 224,
		["name_t"] = "Jumppower",
	},
	[225] = {
		["type"] = 0,
		["exp"] = 5500000000000000000,
		["ability"] = 468,
		["lv"] = 225,
		["name_t"] = "Jumppower",
	},
	[226] = {
		["type"] = 0,
		["exp"] = 6500000000000000000,
		["ability"] = 470,
		["lv"] = 226,
		["name_t"] = "Jumppower",
	},
	[227] = {
		["type"] = 0,
		["exp"] = 7900000000000000000,
		["ability"] = 472,
		["lv"] = 227,
		["name_t"] = "Jumppower",
	},
	[228] = {
		["type"] = 0,
		["exp"] = 9400000000000000000,
		["ability"] = 474,
		["lv"] = 228,
		["name_t"] = "Jumppower",
	},
	[229] = {
		["type"] = 0,
		["exp"] = 11000000000000000000,
		["ability"] = 476,
		["lv"] = 229,
		["name_t"] = "Jumppower",
	},
	[230] = {
		["type"] = 0,
		["exp"] = 14000000000000000000,
		["ability"] = 478,
		["lv"] = 230,
		["name_t"] = "Jumppower",
	},
	[231] = {
		["type"] = 0,
		["exp"] = 16000000000000000000,
		["ability"] = 480,
		["lv"] = 231,
		["name_t"] = "Jumppower",
	},
	[232] = {
		["type"] = 0,
		["exp"] = 20000000000000000000,
		["ability"] = 482,
		["lv"] = 232,
		["name_t"] = "Jumppower",
	},
	[233] = {
		["type"] = 0,
		["exp"] = 23000000000000000000,
		["ability"] = 484,
		["lv"] = 233,
		["name_t"] = "Jumppower",
	},
	[234] = {
		["type"] = 0,
		["exp"] = 28000000000000000000,
		["ability"] = 486,
		["lv"] = 234,
		["name_t"] = "Jumppower",
	},
	[235] = {
		["type"] = 0,
		["exp"] = 34000000000000000000,
		["ability"] = 488,
		["lv"] = 235,
		["name_t"] = "Jumppower",
	},
	[236] = {
		["type"] = 0,
		["exp"] = 41000000000000000000,
		["ability"] = 490,
		["lv"] = 236,
		["name_t"] = "Jumppower",
	},
	[237] = {
		["type"] = 0,
		["exp"] = 49000000000000000000,
		["ability"] = 492,
		["lv"] = 237,
		["name_t"] = "Jumppower",
	},
	[238] = {
		["type"] = 0,
		["exp"] = 58000000000000000000,
		["ability"] = 494,
		["lv"] = 238,
		["name_t"] = "Jumppower",
	},
	[239] = {
		["type"] = 0,
		["exp"] = 70000000000000000000,
		["ability"] = 496,
		["lv"] = 239,
		["name_t"] = "Jumppower",
	},
	[240] = {
		["type"] = 0,
		["exp"] = 84000000000000000000,
		["ability"] = 498,
		["lv"] = 240,
		["name_t"] = "Jumppower",
	},
	[241] = {
		["type"] = 0,
		["exp"] = 100000000000000000000,
		["ability"] = 500,
		["lv"] = 241,
		["name_t"] = "Jumppower",
	},
	[242] = {
		["type"] = 0,
		["exp"] = 120000000000000000000,
		["ability"] = 502,
		["lv"] = 242,
		["name_t"] = "Jumppower",
	},
	[243] = {
		["type"] = 0,
		["exp"] = 150000000000000000000,
		["ability"] = 504,
		["lv"] = 243,
		["name_t"] = "Jumppower",
	},
	[244] = {
		["type"] = 0,
		["exp"] = 170000000000000000000,
		["ability"] = 506,
		["lv"] = 244,
		["name_t"] = "Jumppower",
	},
	[245] = {
		["type"] = 0,
		["exp"] = 210000000000000000000,
		["ability"] = 508,
		["lv"] = 245,
		["name_t"] = "Jumppower",
	},
	[246] = {
		["type"] = 0,
		["exp"] = 250000000000000000000,
		["ability"] = 510,
		["lv"] = 246,
		["name_t"] = "Jumppower",
	},
	[247] = {
		["type"] = 0,
		["exp"] = 300000000000000000000,
		["ability"] = 512,
		["lv"] = 247,
		["name_t"] = "Jumppower",
	},
	[248] = {
		["type"] = 0,
		["exp"] = 360000000000000000000,
		["ability"] = 514,
		["lv"] = 248,
		["name_t"] = "Jumppower",
	},
	[249] = {
		["type"] = 0,
		["exp"] = 430000000000000000000,
		["ability"] = 516,
		["lv"] = 249,
		["name_t"] = "Jumppower",
	},
	[250] = {
		["type"] = 0,
		["exp"] = 520000000000000000000,
		["ability"] = 518,
		["lv"] = 250,
		["name_t"] = "Jumppower",
	},
	[251] = {
		["type"] = 0,
		["exp"] = 620000000000000000000,
		["ability"] = 520,
		["lv"] = 251,
		["name_t"] = "Jumppower",
	},
	[252] = {
		["type"] = 0,
		["exp"] = 750000000000000000000,
		["ability"] = 522,
		["lv"] = 252,
		["name_t"] = "Jumppower",
	},
	[253] = {
		["type"] = 0,
		["exp"] = 900000000000000000000,
		["ability"] = 524,
		["lv"] = 253,
		["name_t"] = "Jumppower",
	},
	[254] = {
		["type"] = 0,
		["exp"] = 1.1e+21,
		["ability"] = 526,
		["lv"] = 254,
		["name_t"] = "Jumppower",
	},
	[255] = {
		["type"] = 0,
		["exp"] = 1.3e+21,
		["ability"] = 528,
		["lv"] = 255,
		["name_t"] = "Jumppower",
	},
	[256] = {
		["type"] = 0,
		["exp"] = 1.6e+21,
		["ability"] = 530,
		["lv"] = 256,
		["name_t"] = "Jumppower",
	},
	[257] = {
		["type"] = 0,
		["exp"] = 1.9e+21,
		["ability"] = 532,
		["lv"] = 257,
		["name_t"] = "Jumppower",
	},
	[258] = {
		["type"] = 0,
		["exp"] = 2.2e+21,
		["ability"] = 534,
		["lv"] = 258,
		["name_t"] = "Jumppower",
	},
	[259] = {
		["type"] = 0,
		["exp"] = 2.7e+21,
		["ability"] = 536,
		["lv"] = 259,
		["name_t"] = "Jumppower",
	},
	[260] = {
		["type"] = 0,
		["exp"] = 3.2e+21,
		["ability"] = 538,
		["lv"] = 260,
		["name_t"] = "Jumppower",
	},
	[261] = {
		["type"] = 0,
		["exp"] = 3.9e+21,
		["ability"] = 540,
		["lv"] = 261,
		["name_t"] = "Jumppower",
	},
	[262] = {
		["type"] = 0,
		["exp"] = 4.6e+21,
		["ability"] = 542,
		["lv"] = 262,
		["name_t"] = "Jumppower",
	},
	[263] = {
		["type"] = 0,
		["exp"] = 5.6e+21,
		["ability"] = 544,
		["lv"] = 263,
		["name_t"] = "Jumppower",
	},
	[264] = {
		["type"] = 0,
		["exp"] = 6.7e+21,
		["ability"] = 546,
		["lv"] = 264,
		["name_t"] = "Jumppower",
	},
	[265] = {
		["type"] = 0,
		["exp"] = 8e+21,
		["ability"] = 548,
		["lv"] = 265,
		["name_t"] = "Jumppower",
	},
	[266] = {
		["type"] = 0,
		["exp"] = 9.6e+21,
		["ability"] = 550,
		["lv"] = 266,
		["name_t"] = "Jumppower",
	},
	[267] = {
		["type"] = 0,
		["exp"] = 1.2e+22,
		["ability"] = 552,
		["lv"] = 267,
		["name_t"] = "Jumppower",
	},
	[268] = {
		["type"] = 0,
		["exp"] = 1.4e+22,
		["ability"] = 554,
		["lv"] = 268,
		["name_t"] = "Jumppower",
	},
	[269] = {
		["type"] = 0,
		["exp"] = 1.7e+22,
		["ability"] = 556,
		["lv"] = 269,
		["name_t"] = "Jumppower",
	},
	[270] = {
		["type"] = 0,
		["exp"] = 2e+22,
		["ability"] = 558,
		["lv"] = 270,
		["name_t"] = "Jumppower",
	},
	[271] = {
		["type"] = 0,
		["exp"] = 2.4e+22,
		["ability"] = 560,
		["lv"] = 271,
		["name_t"] = "Jumppower",
	},
	[272] = {
		["type"] = 0,
		["exp"] = 2.9e+22,
		["ability"] = 562,
		["lv"] = 272,
		["name_t"] = "Jumppower",
	},
	[273] = {
		["type"] = 0,
		["exp"] = 3.4e+22,
		["ability"] = 564,
		["lv"] = 273,
		["name_t"] = "Jumppower",
	},
	[274] = {
		["type"] = 0,
		["exp"] = 4.1e+22,
		["ability"] = 566,
		["lv"] = 274,
		["name_t"] = "Jumppower",
	},
	[275] = {
		["type"] = 0,
		["exp"] = 5e+22,
		["ability"] = 568,
		["lv"] = 275,
		["name_t"] = "Jumppower",
	},
	[276] = {
		["type"] = 0,
		["exp"] = 6e+22,
		["ability"] = 570,
		["lv"] = 276,
		["name_t"] = "Jumppower",
	},
	[277] = {
		["type"] = 0,
		["exp"] = 7.1e+22,
		["ability"] = 572,
		["lv"] = 277,
		["name_t"] = "Jumppower",
	},
	[278] = {
		["type"] = 0,
		["exp"] = 8.6e+22,
		["ability"] = 574,
		["lv"] = 278,
		["name_t"] = "Jumppower",
	},
	[279] = {
		["type"] = 0,
		["exp"] = 1e+23,
		["ability"] = 576,
		["lv"] = 279,
		["name_t"] = "Jumppower",
	},
	[280] = {
		["type"] = 0,
		["exp"] = 1.2e+23,
		["ability"] = 578,
		["lv"] = 280,
		["name_t"] = "Jumppower",
	},
	[281] = {
		["type"] = 0,
		["exp"] = 1.5e+23,
		["ability"] = 580,
		["lv"] = 281,
		["name_t"] = "Jumppower",
	},
	[282] = {
		["type"] = 0,
		["exp"] = 1.8e+23,
		["ability"] = 582,
		["lv"] = 282,
		["name_t"] = "Jumppower",
	},
	[283] = {
		["type"] = 0,
		["exp"] = 2.1e+23,
		["ability"] = 584,
		["lv"] = 283,
		["name_t"] = "Jumppower",
	},
	[284] = {
		["type"] = 0,
		["exp"] = 2.6e+23,
		["ability"] = 586,
		["lv"] = 284,
		["name_t"] = "Jumppower",
	},
	[285] = {
		["type"] = 0,
		["exp"] = 3.1e+23,
		["ability"] = 588,
		["lv"] = 285,
		["name_t"] = "Jumppower",
	},
	[286] = {
		["type"] = 0,
		["exp"] = 3.7e+23,
		["ability"] = 590,
		["lv"] = 286,
		["name_t"] = "Jumppower",
	},
	[287] = {
		["type"] = 0,
		["exp"] = 4.4e+23,
		["ability"] = 592,
		["lv"] = 287,
		["name_t"] = "Jumppower",
	},
	[288] = {
		["type"] = 0,
		["exp"] = 5.3e+23,
		["ability"] = 594,
		["lv"] = 288,
		["name_t"] = "Jumppower",
	},
	[289] = {
		["type"] = 0,
		["exp"] = 6.4e+23,
		["ability"] = 596,
		["lv"] = 289,
		["name_t"] = "Jumppower",
	},
	[290] = {
		["type"] = 0,
		["exp"] = 7.6e+23,
		["ability"] = 598,
		["lv"] = 290,
		["name_t"] = "Jumppower",
	},
	[291] = {
		["type"] = 0,
		["exp"] = 9.2e+23,
		["ability"] = 600,
		["lv"] = 291,
		["name_t"] = "Jumppower",
	},
	[292] = {
		["type"] = 0,
		["exp"] = 1.1e+24,
		["ability"] = 602,
		["lv"] = 292,
		["name_t"] = "Jumppower",
	},
	[293] = {
		["type"] = 0,
		["exp"] = 1.3e+24,
		["ability"] = 604,
		["lv"] = 293,
		["name_t"] = "Jumppower",
	},
	[294] = {
		["type"] = 0,
		["exp"] = 1.6e+24,
		["ability"] = 606,
		["lv"] = 294,
		["name_t"] = "Jumppower",
	},
	[295] = {
		["type"] = 0,
		["exp"] = 1.9e+24,
		["ability"] = 608,
		["lv"] = 295,
		["name_t"] = "Jumppower",
	},
	[296] = {
		["type"] = 0,
		["exp"] = 2.3e+24,
		["ability"] = 610,
		["lv"] = 296,
		["name_t"] = "Jumppower",
	},
	[297] = {
		["type"] = 0,
		["exp"] = 2.7e+24,
		["ability"] = 612,
		["lv"] = 297,
		["name_t"] = "Jumppower",
	},
	[298] = {
		["type"] = 0,
		["exp"] = 3.3e+24,
		["ability"] = 614,
		["lv"] = 298,
		["name_t"] = "Jumppower",
	},
	[299] = {
		["type"] = 0,
		["exp"] = 3.9e+24,
		["ability"] = 616,
		["lv"] = 299,
		["name_t"] = "Jumppower",
	},
	[300] = {
		["type"] = 0,
		["exp"] = 4.7e+24,
		["ability"] = 618,
		["lv"] = 300,
		["name_t"] = "Jumppower",
	},
	[301] = {
		["type"] = 0,
		["exp"] = 5.7e+24,
		["ability"] = 620,
		["lv"] = 301,
		["name_t"] = "Jumppower",
	},
	[302] = {
		["type"] = 0,
		["exp"] = 6.8e+24,
		["ability"] = 622,
		["lv"] = 302,
		["name_t"] = "Jumppower",
	},
	[303] = {
		["type"] = 0,
		["exp"] = 8.2e+24,
		["ability"] = 624,
		["lv"] = 303,
		["name_t"] = "Jumppower",
	},
	[304] = {
		["type"] = 0,
		["exp"] = 9.8e+24,
		["ability"] = 626,
		["lv"] = 304,
		["name_t"] = "Jumppower",
	},
	[305] = {
		["type"] = 0,
		["exp"] = 1.2e+25,
		["ability"] = 628,
		["lv"] = 305,
		["name_t"] = "Jumppower",
	},
	[306] = {
		["type"] = 0,
		["exp"] = 1.4e+25,
		["ability"] = 630,
		["lv"] = 306,
		["name_t"] = "Jumppower",
	},
	[307] = {
		["type"] = 0,
		["exp"] = 1.7e+25,
		["ability"] = 632,
		["lv"] = 307,
		["name_t"] = "Jumppower",
	},
	[308] = {
		["type"] = 0,
		["exp"] = 2e+25,
		["ability"] = 634,
		["lv"] = 308,
		["name_t"] = "Jumppower",
	},
	[309] = {
		["type"] = 0,
		["exp"] = 2.4e+25,
		["ability"] = 636,
		["lv"] = 309,
		["name_t"] = "Jumppower",
	},
	[310] = {
		["type"] = 0,
		["exp"] = 2.9e+25,
		["ability"] = 638,
		["lv"] = 310,
		["name_t"] = "Jumppower",
	},
	[311] = {
		["type"] = 0,
		["exp"] = 3.5e+25,
		["ability"] = 640,
		["lv"] = 311,
		["name_t"] = "Jumppower",
	},
	[312] = {
		["type"] = 0,
		["exp"] = 4.2e+25,
		["ability"] = 642,
		["lv"] = 312,
		["name_t"] = "Jumppower",
	},
	[313] = {
		["type"] = 0,
		["exp"] = 5.1e+25,
		["ability"] = 644,
		["lv"] = 313,
		["name_t"] = "Jumppower",
	},
	[314] = {
		["type"] = 0,
		["exp"] = 6.1e+25,
		["ability"] = 646,
		["lv"] = 314,
		["name_t"] = "Jumppower",
	},
	[315] = {
		["type"] = 0,
		["exp"] = 7.3e+25,
		["ability"] = 648,
		["lv"] = 315,
		["name_t"] = "Jumppower",
	},
	[316] = {
		["type"] = 0,
		["exp"] = 8.8e+25,
		["ability"] = 650,
		["lv"] = 316,
		["name_t"] = "Jumppower",
	},
	[317] = {
		["type"] = 0,
		["exp"] = 1.1e+26,
		["ability"] = 652,
		["lv"] = 317,
		["name_t"] = "Jumppower",
	},
	[318] = {
		["type"] = 0,
		["exp"] = 1.3e+26,
		["ability"] = 654,
		["lv"] = 318,
		["name_t"] = "Jumppower",
	},
	[319] = {
		["type"] = 0,
		["exp"] = 1.5e+26,
		["ability"] = 656,
		["lv"] = 319,
		["name_t"] = "Jumppower",
	},
	[320] = {
		["type"] = 0,
		["exp"] = 1.8e+26,
		["ability"] = 658,
		["lv"] = 320,
		["name_t"] = "Jumppower",
	},
	[321] = {
		["type"] = 0,
		["exp"] = 2.2e+26,
		["ability"] = 660,
		["lv"] = 321,
		["name_t"] = "Jumppower",
	},
	[322] = {
		["type"] = 0,
		["exp"] = 2.6e+26,
		["ability"] = 662,
		["lv"] = 322,
		["name_t"] = "Jumppower",
	},
	[323] = {
		["type"] = 0,
		["exp"] = 3.1e+26,
		["ability"] = 664,
		["lv"] = 323,
		["name_t"] = "Jumppower",
	},
	[324] = {
		["type"] = 0,
		["exp"] = 3.8e+26,
		["ability"] = 666,
		["lv"] = 324,
		["name_t"] = "Jumppower",
	},
	[325] = {
		["type"] = 0,
		["exp"] = 4.5e+26,
		["ability"] = 668,
		["lv"] = 325,
		["name_t"] = "Jumppower",
	},
	[326] = {
		["type"] = 0,
		["exp"] = 5.4e+26,
		["ability"] = 670,
		["lv"] = 326,
		["name_t"] = "Jumppower",
	},
	[327] = {
		["type"] = 0,
		["exp"] = 6.5e+26,
		["ability"] = 672,
		["lv"] = 327,
		["name_t"] = "Jumppower",
	},
	[328] = {
		["type"] = 0,
		["exp"] = 7.8e+26,
		["ability"] = 674,
		["lv"] = 328,
		["name_t"] = "Jumppower",
	},
	[329] = {
		["type"] = 0,
		["exp"] = 9.4e+26,
		["ability"] = 676,
		["lv"] = 329,
		["name_t"] = "Jumppower",
	},
	[330] = {
		["type"] = 0,
		["exp"] = 1.1e+27,
		["ability"] = 678,
		["lv"] = 330,
		["name_t"] = "Jumppower",
	},
	[331] = {
		["type"] = 0,
		["exp"] = 1.3e+27,
		["ability"] = 680,
		["lv"] = 331,
		["name_t"] = "Jumppower",
	},
	[332] = {
		["type"] = 0,
		["exp"] = 1.6e+27,
		["ability"] = 682,
		["lv"] = 332,
		["name_t"] = "Jumppower",
	},
	[333] = {
		["type"] = 0,
		["exp"] = 1.9e+27,
		["ability"] = 684,
		["lv"] = 333,
		["name_t"] = "Jumppower",
	},
	[334] = {
		["type"] = 0,
		["exp"] = 2.3e+27,
		["ability"] = 686,
		["lv"] = 334,
		["name_t"] = "Jumppower",
	},
	[335] = {
		["type"] = 0,
		["exp"] = 2.8e+27,
		["ability"] = 688,
		["lv"] = 335,
		["name_t"] = "Jumppower",
	},
	[336] = {
		["type"] = 0,
		["exp"] = 3.4e+27,
		["ability"] = 690,
		["lv"] = 336,
		["name_t"] = "Jumppower",
	},
	[337] = {
		["type"] = 0,
		["exp"] = 4e+27,
		["ability"] = 692,
		["lv"] = 337,
		["name_t"] = "Jumppower",
	},
	[338] = {
		["type"] = 0,
		["exp"] = 4.8e+27,
		["ability"] = 694,
		["lv"] = 338,
		["name_t"] = "Jumppower",
	},
	[339] = {
		["type"] = 0,
		["exp"] = 5.8e+27,
		["ability"] = 696,
		["lv"] = 339,
		["name_t"] = "Jumppower",
	},
	[340] = {
		["type"] = 0,
		["exp"] = 7e+27,
		["ability"] = 698,
		["lv"] = 340,
		["name_t"] = "Jumppower",
	},
	[341] = {
		["type"] = 0,
		["exp"] = 8.3e+27,
		["ability"] = 700,
		["lv"] = 341,
		["name_t"] = "Jumppower",
	},
	[342] = {
		["type"] = 0,
		["exp"] = 1e+28,
		["ability"] = 702,
		["lv"] = 342,
		["name_t"] = "Jumppower",
	},
	[343] = {
		["type"] = 0,
		["exp"] = 1.2e+28,
		["ability"] = 704,
		["lv"] = 343,
		["name_t"] = "Jumppower",
	},
	[344] = {
		["type"] = 0,
		["exp"] = 1.4e+28,
		["ability"] = 706,
		["lv"] = 344,
		["name_t"] = "Jumppower",
	},
	[345] = {
		["type"] = 0,
		["exp"] = 1.7e+28,
		["ability"] = 708,
		["lv"] = 345,
		["name_t"] = "Jumppower",
	},
	[346] = {
		["type"] = 0,
		["exp"] = 2.1e+28,
		["ability"] = 710,
		["lv"] = 346,
		["name_t"] = "Jumppower",
	},
	[347] = {
		["type"] = 0,
		["exp"] = 2.5e+28,
		["ability"] = 712,
		["lv"] = 347,
		["name_t"] = "Jumppower",
	},
	[348] = {
		["type"] = 0,
		["exp"] = 3e+28,
		["ability"] = 714,
		["lv"] = 348,
		["name_t"] = "Jumppower",
	},
	[349] = {
		["type"] = 0,
		["exp"] = 3.6e+28,
		["ability"] = 716,
		["lv"] = 349,
		["name_t"] = "Jumppower",
	},
	[350] = {
		["type"] = 0,
		["exp"] = 4.3e+28,
		["ability"] = 718,
		["lv"] = 350,
		["name_t"] = "Jumppower",
	},
	[351] = {
		["type"] = 0,
		["exp"] = 5.2e+28,
		["ability"] = 720,
		["lv"] = 351,
		["name_t"] = "Jumppower",
	},
	[352] = {
		["type"] = 0,
		["exp"] = 6.2e+28,
		["ability"] = 722,
		["lv"] = 352,
		["name_t"] = "Jumppower",
	},
	[353] = {
		["type"] = 0,
		["exp"] = 7.4e+28,
		["ability"] = 724,
		["lv"] = 353,
		["name_t"] = "Jumppower",
	},
	[354] = {
		["type"] = 0,
		["exp"] = 8.9e+28,
		["ability"] = 726,
		["lv"] = 354,
		["name_t"] = "Jumppower",
	},
	[355] = {
		["type"] = 0,
		["exp"] = 1.1e+29,
		["ability"] = 728,
		["lv"] = 355,
		["name_t"] = "Jumppower",
	},
	[356] = {
		["type"] = 0,
		["exp"] = 1.3e+29,
		["ability"] = 730,
		["lv"] = 356,
		["name_t"] = "Jumppower",
	},
	[357] = {
		["type"] = 0,
		["exp"] = 1.5e+29,
		["ability"] = 732,
		["lv"] = 357,
		["name_t"] = "Jumppower",
	},
	[358] = {
		["type"] = 0,
		["exp"] = 1.9e+29,
		["ability"] = 734,
		["lv"] = 358,
		["name_t"] = "Jumppower",
	},
	[359] = {
		["type"] = 0,
		["exp"] = 2.2e+29,
		["ability"] = 736,
		["lv"] = 359,
		["name_t"] = "Jumppower",
	},
	[360] = {
		["type"] = 0,
		["exp"] = 2.7e+29,
		["ability"] = 738,
		["lv"] = 360,
		["name_t"] = "Jumppower",
	},
	[361] = {
		["type"] = 0,
		["exp"] = 3.2e+29,
		["ability"] = 740,
		["lv"] = 361,
		["name_t"] = "Jumppower",
	},
	[362] = {
		["type"] = 0,
		["exp"] = 3.8e+29,
		["ability"] = 742,
		["lv"] = 362,
		["name_t"] = "Jumppower",
	},
	[363] = {
		["type"] = 0,
		["exp"] = 4.6e+29,
		["ability"] = 744,
		["lv"] = 363,
		["name_t"] = "Jumppower",
	},
	[364] = {
		["type"] = 0,
		["exp"] = 5.5e+29,
		["ability"] = 746,
		["lv"] = 364,
		["name_t"] = "Jumppower",
	},
	[365] = {
		["type"] = 0,
		["exp"] = 6.6e+29,
		["ability"] = 748,
		["lv"] = 365,
		["name_t"] = "Jumppower",
	},
	[366] = {
		["type"] = 0,
		["exp"] = 8e+29,
		["ability"] = 750,
		["lv"] = 366,
		["name_t"] = "Jumppower",
	},
	[367] = {
		["type"] = 0,
		["exp"] = 9.6e+29,
		["ability"] = 752,
		["lv"] = 367,
		["name_t"] = "Jumppower",
	},
	[368] = {
		["type"] = 0,
		["exp"] = 1.1e+30,
		["ability"] = 754,
		["lv"] = 368,
		["name_t"] = "Jumppower",
	},
	[369] = {
		["type"] = 0,
		["exp"] = 1.4e+30,
		["ability"] = 756,
		["lv"] = 369,
		["name_t"] = "Jumppower",
	},
	[370] = {
		["type"] = 0,
		["exp"] = 1.7e+30,
		["ability"] = 758,
		["lv"] = 370,
		["name_t"] = "Jumppower",
	},
	[371] = {
		["type"] = 0,
		["exp"] = 2e+30,
		["ability"] = 760,
		["lv"] = 371,
		["name_t"] = "Jumppower",
	},
	[372] = {
		["type"] = 0,
		["exp"] = 2.4e+30,
		["ability"] = 762,
		["lv"] = 372,
		["name_t"] = "Jumppower",
	},
	[373] = {
		["type"] = 0,
		["exp"] = 2.9e+30,
		["ability"] = 764,
		["lv"] = 373,
		["name_t"] = "Jumppower",
	},
	[374] = {
		["type"] = 0,
		["exp"] = 3.4e+30,
		["ability"] = 766,
		["lv"] = 374,
		["name_t"] = "Jumppower",
	},
	[375] = {
		["type"] = 0,
		["exp"] = 4.1e+30,
		["ability"] = 768,
		["lv"] = 375,
		["name_t"] = "Jumppower",
	},
	[376] = {
		["type"] = 0,
		["exp"] = 4.9e+30,
		["ability"] = 770,
		["lv"] = 376,
		["name_t"] = "Jumppower",
	},
	[377] = {
		["type"] = 0,
		["exp"] = 5.9e+30,
		["ability"] = 772,
		["lv"] = 377,
		["name_t"] = "Jumppower",
	},
	[378] = {
		["type"] = 0,
		["exp"] = 7.1e+30,
		["ability"] = 774,
		["lv"] = 378,
		["name_t"] = "Jumppower",
	},
	[379] = {
		["type"] = 0,
		["exp"] = 8.5e+30,
		["ability"] = 776,
		["lv"] = 379,
		["name_t"] = "Jumppower",
	},
	[380] = {
		["type"] = 0,
		["exp"] = 1e+31,
		["ability"] = 778,
		["lv"] = 380,
		["name_t"] = "Jumppower",
	},
	[381] = {
		["type"] = 0,
		["exp"] = 1.2e+31,
		["ability"] = 780,
		["lv"] = 381,
		["name_t"] = "Jumppower",
	},
	[382] = {
		["type"] = 0,
		["exp"] = 1.5e+31,
		["ability"] = 782,
		["lv"] = 382,
		["name_t"] = "Jumppower",
	},
	[383] = {
		["type"] = 0,
		["exp"] = 1.8e+31,
		["ability"] = 784,
		["lv"] = 383,
		["name_t"] = "Jumppower",
	},
	[384] = {
		["type"] = 0,
		["exp"] = 2.1e+31,
		["ability"] = 786,
		["lv"] = 384,
		["name_t"] = "Jumppower",
	},
	[385] = {
		["type"] = 0,
		["exp"] = 2.5e+31,
		["ability"] = 788,
		["lv"] = 385,
		["name_t"] = "Jumppower",
	},
	[386] = {
		["type"] = 0,
		["exp"] = 3.1e+31,
		["ability"] = 790,
		["lv"] = 386,
		["name_t"] = "Jumppower",
	},
	[387] = {
		["type"] = 0,
		["exp"] = 3.7e+31,
		["ability"] = 792,
		["lv"] = 387,
		["name_t"] = "Jumppower",
	},
	[388] = {
		["type"] = 0,
		["exp"] = 4.4e+31,
		["ability"] = 794,
		["lv"] = 388,
		["name_t"] = "Jumppower",
	},
	[389] = {
		["type"] = 0,
		["exp"] = 5.3e+31,
		["ability"] = 796,
		["lv"] = 389,
		["name_t"] = "Jumppower",
	},
	[390] = {
		["type"] = 0,
		["exp"] = 6.3e+31,
		["ability"] = 798,
		["lv"] = 390,
		["name_t"] = "Jumppower",
	},
	[391] = {
		["type"] = 0,
		["exp"] = 7.6e+31,
		["ability"] = 800,
		["lv"] = 391,
		["name_t"] = "Jumppower",
	},
	[392] = {
		["type"] = 0,
		["exp"] = 9.1e+31,
		["ability"] = 802,
		["lv"] = 392,
		["name_t"] = "Jumppower",
	},
	[393] = {
		["type"] = 0,
		["exp"] = 1.1e+32,
		["ability"] = 804,
		["lv"] = 393,
		["name_t"] = "Jumppower",
	},
	[394] = {
		["type"] = 0,
		["exp"] = 1.3e+32,
		["ability"] = 806,
		["lv"] = 394,
		["name_t"] = "Jumppower",
	},
	[395] = {
		["type"] = 0,
		["exp"] = 1.6e+32,
		["ability"] = 808,
		["lv"] = 395,
		["name_t"] = "Jumppower",
	},
	[396] = {
		["type"] = 0,
		["exp"] = 1.9e+32,
		["ability"] = 810,
		["lv"] = 396,
		["name_t"] = "Jumppower",
	},
	[397] = {
		["type"] = 0,
		["exp"] = 2.3e+32,
		["ability"] = 812,
		["lv"] = 397,
		["name_t"] = "Jumppower",
	},
	[398] = {
		["type"] = 0,
		["exp"] = 2.7e+32,
		["ability"] = 814,
		["lv"] = 398,
		["name_t"] = "Jumppower",
	},
	[399] = {
		["type"] = 0,
		["exp"] = 3.3e+32,
		["ability"] = 816,
		["lv"] = 399,
		["name_t"] = "Jumppower",
	},
	[400] = {
		["type"] = 0,
		["exp"] = 3.9e+32,
		["ability"] = 818,
		["lv"] = 400,
		["name_t"] = "Jumppower",
	},
	[401] = {
		["type"] = 0,
		["exp"] = 4.7e+32,
		["ability"] = 820,
		["lv"] = 401,
		["name_t"] = "Jumppower",
	},
	[402] = {
		["type"] = 0,
		["exp"] = 5.6e+32,
		["ability"] = 822,
		["lv"] = 402,
		["name_t"] = "Jumppower",
	},
	[403] = {
		["type"] = 0,
		["exp"] = 6.8e+32,
		["ability"] = 824,
		["lv"] = 403,
		["name_t"] = "Jumppower",
	},
	[404] = {
		["type"] = 0,
		["exp"] = 8.1e+32,
		["ability"] = 826,
		["lv"] = 404,
		["name_t"] = "Jumppower",
	},
	[405] = {
		["type"] = 0,
		["exp"] = 9.8e+32,
		["ability"] = 828,
		["lv"] = 405,
		["name_t"] = "Jumppower",
	},
	[406] = {
		["type"] = 0,
		["exp"] = 1.2e+33,
		["ability"] = 830,
		["lv"] = 406,
		["name_t"] = "Jumppower",
	},
	[407] = {
		["type"] = 0,
		["exp"] = 1.4e+33,
		["ability"] = 832,
		["lv"] = 407,
		["name_t"] = "Jumppower",
	},
	[408] = {
		["type"] = 0,
		["exp"] = 1.7e+33,
		["ability"] = 834,
		["lv"] = 408,
		["name_t"] = "Jumppower",
	},
	[409] = {
		["type"] = 0,
		["exp"] = 2e+33,
		["ability"] = 836,
		["lv"] = 409,
		["name_t"] = "Jumppower",
	},
	[410] = {
		["type"] = 0,
		["exp"] = 2.4e+33,
		["ability"] = 838,
		["lv"] = 410,
		["name_t"] = "Jumppower",
	},
	[411] = {
		["type"] = 0,
		["exp"] = 2.9e+33,
		["ability"] = 840,
		["lv"] = 411,
		["name_t"] = "Jumppower",
	},
	[412] = {
		["type"] = 0,
		["exp"] = 3.5e+33,
		["ability"] = 842,
		["lv"] = 412,
		["name_t"] = "Jumppower",
	},
	[413] = {
		["type"] = 0,
		["exp"] = 4.2e+33,
		["ability"] = 844,
		["lv"] = 413,
		["name_t"] = "Jumppower",
	},
	[414] = {
		["type"] = 0,
		["exp"] = 5e+33,
		["ability"] = 846,
		["lv"] = 414,
		["name_t"] = "Jumppower",
	},
	[415] = {
		["type"] = 0,
		["exp"] = 6e+33,
		["ability"] = 848,
		["lv"] = 415,
		["name_t"] = "Jumppower",
	},
	[416] = {
		["type"] = 0,
		["exp"] = 7.2e+33,
		["ability"] = 850,
		["lv"] = 416,
		["name_t"] = "Jumppower",
	},
	[417] = {
		["type"] = 0,
		["exp"] = 8.7e+33,
		["ability"] = 852,
		["lv"] = 417,
		["name_t"] = "Jumppower",
	},
	[418] = {
		["type"] = 0,
		["exp"] = 1e+34,
		["ability"] = 854,
		["lv"] = 418,
		["name_t"] = "Jumppower",
	},
	[419] = {
		["type"] = 0,
		["exp"] = 1.3e+34,
		["ability"] = 856,
		["lv"] = 419,
		["name_t"] = "Jumppower",
	},
	[420] = {
		["type"] = 0,
		["exp"] = 1.5e+34,
		["ability"] = 858,
		["lv"] = 420,
		["name_t"] = "Jumppower",
	},
	[421] = {
		["type"] = 0,
		["exp"] = 1.8e+34,
		["ability"] = 860,
		["lv"] = 421,
		["name_t"] = "Jumppower",
	},
	[422] = {
		["type"] = 0,
		["exp"] = 2.2e+34,
		["ability"] = 862,
		["lv"] = 422,
		["name_t"] = "Jumppower",
	},
	[423] = {
		["type"] = 0,
		["exp"] = 2.6e+34,
		["ability"] = 864,
		["lv"] = 423,
		["name_t"] = "Jumppower",
	},
	[424] = {
		["type"] = 0,
		["exp"] = 3.1e+34,
		["ability"] = 866,
		["lv"] = 424,
		["name_t"] = "Jumppower",
	},
	[425] = {
		["type"] = 0,
		["exp"] = 3.7e+34,
		["ability"] = 868,
		["lv"] = 425,
		["name_t"] = "Jumppower",
	},
	[426] = {
		["type"] = 0,
		["exp"] = 4.5e+34,
		["ability"] = 870,
		["lv"] = 426,
		["name_t"] = "Jumppower",
	},
	[427] = {
		["type"] = 0,
		["exp"] = 5.4e+34,
		["ability"] = 872,
		["lv"] = 427,
		["name_t"] = "Jumppower",
	},
	[428] = {
		["type"] = 0,
		["exp"] = 6.5e+34,
		["ability"] = 874,
		["lv"] = 428,
		["name_t"] = "Jumppower",
	},
	[429] = {
		["type"] = 0,
		["exp"] = 7.8e+34,
		["ability"] = 876,
		["lv"] = 429,
		["name_t"] = "Jumppower",
	},
	[430] = {
		["type"] = 0,
		["exp"] = 9.3e+34,
		["ability"] = 878,
		["lv"] = 430,
		["name_t"] = "Jumppower",
	},
	[431] = {
		["type"] = 0,
		["exp"] = 1.1e+35,
		["ability"] = 880,
		["lv"] = 431,
		["name_t"] = "Jumppower",
	},
	[432] = {
		["type"] = 0,
		["exp"] = 1.3e+35,
		["ability"] = 882,
		["lv"] = 432,
		["name_t"] = "Jumppower",
	},
	[433] = {
		["type"] = 0,
		["exp"] = 1.6e+35,
		["ability"] = 884,
		["lv"] = 433,
		["name_t"] = "Jumppower",
	},
	[434] = {
		["type"] = 0,
		["exp"] = 1.9e+35,
		["ability"] = 886,
		["lv"] = 434,
		["name_t"] = "Jumppower",
	},
	[435] = {
		["type"] = 0,
		["exp"] = 2.3e+35,
		["ability"] = 888,
		["lv"] = 435,
		["name_t"] = "Jumppower",
	},
	[436] = {
		["type"] = 0,
		["exp"] = 2.8e+35,
		["ability"] = 890,
		["lv"] = 436,
		["name_t"] = "Jumppower",
	},
	[437] = {
		["type"] = 0,
		["exp"] = 3.3e+35,
		["ability"] = 892,
		["lv"] = 437,
		["name_t"] = "Jumppower",
	},
}