return {
	[1] = {
		["type"] = 1,
		["exp"] = 10,
		["ability"] = 20,
		["lv"] = 1,
		["name_t"] = "Jumppower",
	},
	[2] = {
		["type"] = 1,
		["exp"] = 12,
		["ability"] = 22,
		["lv"] = 2,
		["name_t"] = "Jumppower",
	},
	[3] = {
		["type"] = 1,
		["exp"] = 14,
		["ability"] = 24,
		["lv"] = 3,
		["name_t"] = "Jumppower",
	},
	[4] = {
		["type"] = 1,
		["exp"] = 17,
		["ability"] = 26,
		["lv"] = 4,
		["name_t"] = "Jumppower",
	},
	[5] = {
		["type"] = 1,
		["exp"] = 21,
		["ability"] = 28,
		["lv"] = 5,
		["name_t"] = "Jumppower",
	},
	[6] = {
		["type"] = 1,
		["exp"] = 25,
		["ability"] = 30,
		["lv"] = 6,
		["name_t"] = "Jumppower",
	},
	[7] = {
		["type"] = 1,
		["exp"] = 30,
		["ability"] = 32,
		["lv"] = 7,
		["name_t"] = "Jumppower",
	},
	[8] = {
		["type"] = 1,
		["exp"] = 36,
		["ability"] = 34,
		["lv"] = 8,
		["name_t"] = "Jumppower",
	},
	[9] = {
		["type"] = 1,
		["exp"] = 43,
		["ability"] = 36,
		["lv"] = 9,
		["name_t"] = "Jumppower",
	},
	[10] = {
		["type"] = 1,
		["exp"] = 52,
		["ability"] = 38,
		["lv"] = 10,
		["name_t"] = "Jumppower",
	},
	[11] = {
		["type"] = 1,
		["exp"] = 62,
		["ability"] = 40,
		["lv"] = 11,
		["name_t"] = "Jumppower",
	},
	[12] = {
		["type"] = 1,
		["exp"] = 74,
		["ability"] = 42,
		["lv"] = 12,
		["name_t"] = "Jumppower",
	},
	[13] = {
		["type"] = 1,
		["exp"] = 89,
		["ability"] = 44,
		["lv"] = 13,
		["name_t"] = "Jumppower",
	},
	[14] = {
		["type"] = 1,
		["exp"] = 110,
		["ability"] = 46,
		["lv"] = 14,
		["name_t"] = "Jumppower",
	},
	[15] = {
		["type"] = 1,
		["exp"] = 130,
		["ability"] = 48,
		["lv"] = 15,
		["name_t"] = "Jumppower",
	},
	[16] = {
		["type"] = 1,
		["exp"] = 150,
		["ability"] = 50,
		["lv"] = 16,
		["name_t"] = "Jumppower",
	},
	[17] = {
		["type"] = 1,
		["exp"] = 180,
		["ability"] = 52,
		["lv"] = 17,
		["name_t"] = "Jumppower",
	},
	[18] = {
		["type"] = 1,
		["exp"] = 220,
		["ability"] = 54,
		["lv"] = 18,
		["name_t"] = "Jumppower",
	},
	[19] = {
		["type"] = 1,
		["exp"] = 270,
		["ability"] = 56,
		["lv"] = 19,
		["name_t"] = "Jumppower",
	},
	[20] = {
		["type"] = 1,
		["exp"] = 320,
		["ability"] = 58,
		["lv"] = 20,
		["name_t"] = "Jumppower",
	},
	[21] = {
		["type"] = 1,
		["exp"] = 380,
		["ability"] = 60,
		["lv"] = 21,
		["name_t"] = "Jumppower",
	},
	[22] = {
		["type"] = 1,
		["exp"] = 460,
		["ability"] = 62,
		["lv"] = 22,
		["name_t"] = "Jumppower",
	},
	[23] = {
		["type"] = 1,
		["exp"] = 550,
		["ability"] = 64,
		["lv"] = 23,
		["name_t"] = "Jumppower",
	},
	[24] = {
		["type"] = 1,
		["exp"] = 660,
		["ability"] = 66,
		["lv"] = 24,
		["name_t"] = "Jumppower",
	},
	[25] = {
		["type"] = 1,
		["exp"] = 790,
		["ability"] = 68,
		["lv"] = 25,
		["name_t"] = "Jumppower",
	},
	[26] = {
		["type"] = 1,
		["exp"] = 950,
		["ability"] = 70,
		["lv"] = 26,
		["name_t"] = "Jumppower",
	},
	[27] = {
		["type"] = 1,
		["exp"] = 1100,
		["ability"] = 72,
		["lv"] = 27,
		["name_t"] = "Jumppower",
	},
	[28] = {
		["type"] = 1,
		["exp"] = 1400,
		["ability"] = 74,
		["lv"] = 28,
		["name_t"] = "Jumppower",
	},
	[29] = {
		["type"] = 1,
		["exp"] = 1600,
		["ability"] = 76,
		["lv"] = 29,
		["name_t"] = "Jumppower",
	},
	[30] = {
		["type"] = 1,
		["exp"] = 2000,
		["ability"] = 78,
		["lv"] = 30,
		["name_t"] = "Jumppower",
	},
	[31] = {
		["type"] = 1,
		["exp"] = 2400,
		["ability"] = 80,
		["lv"] = 31,
		["name_t"] = "Jumppower",
	},
	[32] = {
		["type"] = 1,
		["exp"] = 2800,
		["ability"] = 82,
		["lv"] = 32,
		["name_t"] = "Jumppower",
	},
	[33] = {
		["type"] = 1,
		["exp"] = 3400,
		["ability"] = 84,
		["lv"] = 33,
		["name_t"] = "Jumppower",
	},
	[34] = {
		["type"] = 1,
		["exp"] = 4100,
		["ability"] = 86,
		["lv"] = 34,
		["name_t"] = "Jumppower",
	},
	[35] = {
		["type"] = 1,
		["exp"] = 4900,
		["ability"] = 88,
		["lv"] = 35,
		["name_t"] = "Jumppower",
	},
	[36] = {
		["type"] = 1,
		["exp"] = 5900,
		["ability"] = 90,
		["lv"] = 36,
		["name_t"] = "Jumppower",
	},
	[37] = {
		["type"] = 1,
		["exp"] = 7100,
		["ability"] = 92,
		["lv"] = 37,
		["name_t"] = "Jumppower",
	},
	[38] = {
		["type"] = 1,
		["exp"] = 8500,
		["ability"] = 94,
		["lv"] = 38,
		["name_t"] = "Jumppower",
	},
	[39] = {
		["type"] = 1,
		["exp"] = 10000,
		["ability"] = 96,
		["lv"] = 39,
		["name_t"] = "Jumppower",
	},
	[40] = {
		["type"] = 1,
		["exp"] = 12000,
		["ability"] = 98,
		["lv"] = 40,
		["name_t"] = "Jumppower",
	},
	[41] = {
		["type"] = 1,
		["exp"] = 15000,
		["ability"] = 100,
		["lv"] = 41,
		["name_t"] = "Jumppower",
	},
	[42] = {
		["type"] = 1,
		["exp"] = 18000,
		["ability"] = 102,
		["lv"] = 42,
		["name_t"] = "Jumppower",
	},
	[43] = {
		["type"] = 1,
		["exp"] = 21000,
		["ability"] = 104,
		["lv"] = 43,
		["name_t"] = "Jumppower",
	},
	[44] = {
		["type"] = 1,
		["exp"] = 25000,
		["ability"] = 106,
		["lv"] = 44,
		["name_t"] = "Jumppower",
	},
	[45] = {
		["type"] = 1,
		["exp"] = 30000,
		["ability"] = 108,
		["lv"] = 45,
		["name_t"] = "Jumppower",
	},
	[46] = {
		["type"] = 1,
		["exp"] = 37000,
		["ability"] = 110,
		["lv"] = 46,
		["name_t"] = "Jumppower",
	},
	[47] = {
		["type"] = 1,
		["exp"] = 44000,
		["ability"] = 112,
		["lv"] = 47,
		["name_t"] = "Jumppower",
	},
	[48] = {
		["type"] = 1,
		["exp"] = 53000,
		["ability"] = 114,
		["lv"] = 48,
		["name_t"] = "Jumppower",
	},
	[49] = {
		["type"] = 1,
		["exp"] = 63000,
		["ability"] = 116,
		["lv"] = 49,
		["name_t"] = "Jumppower",
	},
	[50] = {
		["type"] = 1,
		["exp"] = 76000,
		["ability"] = 118,
		["lv"] = 50,
		["name_t"] = "Jumppower",
	},
	[51] = {
		["type"] = 1,
		["exp"] = 91000,
		["ability"] = 120,
		["lv"] = 51,
		["name_t"] = "Jumppower",
	},
	[52] = {
		["type"] = 1,
		["exp"] = 110000,
		["ability"] = 122,
		["lv"] = 52,
		["name_t"] = "Jumppower",
	},
	[53] = {
		["type"] = 1,
		["exp"] = 130000,
		["ability"] = 124,
		["lv"] = 53,
		["name_t"] = "Jumppower",
	},
	[54] = {
		["type"] = 1,
		["exp"] = 160000,
		["ability"] = 126,
		["lv"] = 54,
		["name_t"] = "Jumppower",
	},
	[55] = {
		["type"] = 1,
		["exp"] = 190000,
		["ability"] = 128,
		["lv"] = 55,
		["name_t"] = "Jumppower",
	},
	[56] = {
		["type"] = 1,
		["exp"] = 230000,
		["ability"] = 130,
		["lv"] = 56,
		["name_t"] = "Jumppower",
	},
	[57] = {
		["type"] = 1,
		["exp"] = 270000,
		["ability"] = 132,
		["lv"] = 57,
		["name_t"] = "Jumppower",
	},
	[58] = {
		["type"] = 1,
		["exp"] = 330000,
		["ability"] = 134,
		["lv"] = 58,
		["name_t"] = "Jumppower",
	},
	[59] = {
		["type"] = 1,
		["exp"] = 390000,
		["ability"] = 136,
		["lv"] = 59,
		["name_t"] = "Jumppower",
	},
	[60] = {
		["type"] = 1,
		["exp"] = 470000,
		["ability"] = 138,
		["lv"] = 60,
		["name_t"] = "Jumppower",
	},
	[61] = {
		["type"] = 1,
		["exp"] = 560000,
		["ability"] = 140,
		["lv"] = 61,
		["name_t"] = "Jumppower",
	},
	[62] = {
		["type"] = 1,
		["exp"] = 680000,
		["ability"] = 142,
		["lv"] = 62,
		["name_t"] = "Jumppower",
	},
	[63] = {
		["type"] = 1,
		["exp"] = 810000,
		["ability"] = 144,
		["lv"] = 63,
		["name_t"] = "Jumppower",
	},
	[64] = {
		["type"] = 1,
		["exp"] = 970000,
		["ability"] = 146,
		["lv"] = 64,
		["name_t"] = "Jumppower",
	},
	[65] = {
		["type"] = 1,
		["exp"] = 1200000,
		["ability"] = 148,
		["lv"] = 65,
		["name_t"] = "Jumppower",
	},
	[66] = {
		["type"] = 1,
		["exp"] = 1400000,
		["ability"] = 150,
		["lv"] = 66,
		["name_t"] = "Jumppower",
	},
	[67] = {
		["type"] = 1,
		["exp"] = 1700000,
		["ability"] = 152,
		["lv"] = 67,
		["name_t"] = "Jumppower",
	},
	[68] = {
		["type"] = 1,
		["exp"] = 2000000,
		["ability"] = 154,
		["lv"] = 68,
		["name_t"] = "Jumppower",
	},
	[69] = {
		["type"] = 1,
		["exp"] = 2400000,
		["ability"] = 156,
		["lv"] = 69,
		["name_t"] = "Jumppower",
	},
	[70] = {
		["type"] = 1,
		["exp"] = 2900000,
		["ability"] = 158,
		["lv"] = 70,
		["name_t"] = "Jumppower",
	},
	[71] = {
		["type"] = 1,
		["exp"] = 3500000,
		["ability"] = 160,
		["lv"] = 71,
		["name_t"] = "Jumppower",
	},
	[72] = {
		["type"] = 1,
		["exp"] = 4200000,
		["ability"] = 162,
		["lv"] = 72,
		["name_t"] = "Jumppower",
	},
	[73] = {
		["type"] = 1,
		["exp"] = 5000000,
		["ability"] = 164,
		["lv"] = 73,
		["name_t"] = "Jumppower",
	},
	[74] = {
		["type"] = 1,
		["exp"] = 6000000,
		["ability"] = 166,
		["lv"] = 74,
		["name_t"] = "Jumppower",
	},
	[75] = {
		["type"] = 1,
		["exp"] = 7200000,
		["ability"] = 168,
		["lv"] = 75,
		["name_t"] = "Jumppower",
	},
	[76] = {
		["type"] = 1,
		["exp"] = 8700000,
		["ability"] = 170,
		["lv"] = 76,
		["name_t"] = "Jumppower",
	},
	[77] = {
		["type"] = 1,
		["exp"] = 10000000,
		["ability"] = 172,
		["lv"] = 77,
		["name_t"] = "Jumppower",
	},
	[78] = {
		["type"] = 1,
		["exp"] = 13000000,
		["ability"] = 174,
		["lv"] = 78,
		["name_t"] = "Jumppower",
	},
	[79] = {
		["type"] = 1,
		["exp"] = 15000000,
		["ability"] = 176,
		["lv"] = 79,
		["name_t"] = "Jumppower",
	},
	[80] = {
		["type"] = 1,
		["exp"] = 18000000,
		["ability"] = 178,
		["lv"] = 80,
		["name_t"] = "Jumppower",
	},
	[81] = {
		["type"] = 1,
		["exp"] = 22000000,
		["ability"] = 180,
		["lv"] = 81,
		["name_t"] = "Jumppower",
	},
	[82] = {
		["type"] = 1,
		["exp"] = 26000000,
		["ability"] = 182,
		["lv"] = 82,
		["name_t"] = "Jumppower",
	},
	[83] = {
		["type"] = 1,
		["exp"] = 31000000,
		["ability"] = 184,
		["lv"] = 83,
		["name_t"] = "Jumppower",
	},
	[84] = {
		["type"] = 1,
		["exp"] = 37000000,
		["ability"] = 186,
		["lv"] = 84,
		["name_t"] = "Jumppower",
	},
	[85] = {
		["type"] = 1,
		["exp"] = 45000000,
		["ability"] = 188,
		["lv"] = 85,
		["name_t"] = "Jumppower",
	},
	[86] = {
		["type"] = 1,
		["exp"] = 54000000,
		["ability"] = 190,
		["lv"] = 86,
		["name_t"] = "Jumppower",
	},
	[87] = {
		["type"] = 1,
		["exp"] = 65000000,
		["ability"] = 192,
		["lv"] = 87,
		["name_t"] = "Jumppower",
	},
	[88] = {
		["type"] = 1,
		["exp"] = 77000000,
		["ability"] = 194,
		["lv"] = 88,
		["name_t"] = "Jumppower",
	},
	[89] = {
		["type"] = 1,
		["exp"] = 93000000,
		["ability"] = 196,
		["lv"] = 89,
		["name_t"] = "Jumppower",
	},
	[90] = {
		["type"] = 1,
		["exp"] = 110000000,
		["ability"] = 198,
		["lv"] = 90,
		["name_t"] = "Jumppower",
	},
	[91] = {
		["type"] = 1,
		["exp"] = 130000000,
		["ability"] = 200,
		["lv"] = 91,
		["name_t"] = "Jumppower",
	},
	[92] = {
		["type"] = 1,
		["exp"] = 160000000,
		["ability"] = 202,
		["lv"] = 92,
		["name_t"] = "Jumppower",
	},
	[93] = {
		["type"] = 1,
		["exp"] = 190000000,
		["ability"] = 204,
		["lv"] = 93,
		["name_t"] = "Jumppower",
	},
	[94] = {
		["type"] = 1,
		["exp"] = 230000000,
		["ability"] = 206,
		["lv"] = 94,
		["name_t"] = "Jumppower",
	},
	[95] = {
		["type"] = 1,
		["exp"] = 280000000,
		["ability"] = 208,
		["lv"] = 95,
		["name_t"] = "Jumppower",
	},
	[96] = {
		["type"] = 1,
		["exp"] = 330000000,
		["ability"] = 210,
		["lv"] = 96,
		["name_t"] = "Jumppower",
	},
	[97] = {
		["type"] = 1,
		["exp"] = 400000000,
		["ability"] = 212,
		["lv"] = 97,
		["name_t"] = "Jumppower",
	},
	[98] = {
		["type"] = 1,
		["exp"] = 480000000,
		["ability"] = 214,
		["lv"] = 98,
		["name_t"] = "Jumppower",
	},
	[99] = {
		["type"] = 1,
		["exp"] = 580000000,
		["ability"] = 216,
		["lv"] = 99,
		["name_t"] = "Jumppower",
	},
	[100] = {
		["type"] = 1,
		["exp"] = 690000000,
		["ability"] = 218,
		["lv"] = 100,
		["name_t"] = "Jumppower",
	},
	[101] = {
		["type"] = 1,
		["exp"] = 830000000,
		["ability"] = 220,
		["lv"] = 101,
		["name_t"] = "Jumppower",
	},
	[102] = {
		["type"] = 1,
		["exp"] = 990000000,
		["ability"] = 222,
		["lv"] = 102,
		["name_t"] = "Jumppower",
	},
	[103] = {
		["type"] = 1,
		["exp"] = 1200000000,
		["ability"] = 224,
		["lv"] = 103,
		["name_t"] = "Jumppower",
	},
	[104] = {
		["type"] = 1,
		["exp"] = 1400000000,
		["ability"] = 226,
		["lv"] = 104,
		["name_t"] = "Jumppower",
	},
	[105] = {
		["type"] = 1,
		["exp"] = 1700000000,
		["ability"] = 228,
		["lv"] = 105,
		["name_t"] = "Jumppower",
	},
	[106] = {
		["type"] = 1,
		["exp"] = 2100000000,
		["ability"] = 230,
		["lv"] = 106,
		["name_t"] = "Jumppower",
	},
	[107] = {
		["type"] = 1,
		["exp"] = 2500000000,
		["ability"] = 232,
		["lv"] = 107,
		["name_t"] = "Jumppower",
	},
	[108] = {
		["type"] = 1,
		["exp"] = 3000000000,
		["ability"] = 234,
		["lv"] = 108,
		["name_t"] = "Jumppower",
	},
	[109] = {
		["type"] = 1,
		["exp"] = 3600000000,
		["ability"] = 236,
		["lv"] = 109,
		["name_t"] = "Jumppower",
	},
	[110] = {
		["type"] = 1,
		["exp"] = 4300000000,
		["ability"] = 238,
		["lv"] = 110,
		["name_t"] = "Jumppower",
	},
	[111] = {
		["type"] = 1,
		["exp"] = 5100000000,
		["ability"] = 240,
		["lv"] = 111,
		["name_t"] = "Jumppower",
	},
	[112] = {
		["type"] = 1,
		["exp"] = 6200000000,
		["ability"] = 242,
		["lv"] = 112,
		["name_t"] = "Jumppower",
	},
	[113] = {
		["type"] = 1,
		["exp"] = 7400000000,
		["ability"] = 244,
		["lv"] = 113,
		["name_t"] = "Jumppower",
	},
	[114] = {
		["type"] = 1,
		["exp"] = 8900000000,
		["ability"] = 246,
		["lv"] = 114,
		["name_t"] = "Jumppower",
	},
	[115] = {
		["type"] = 1,
		["exp"] = 11000000000,
		["ability"] = 248,
		["lv"] = 115,
		["name_t"] = "Jumppower",
	},
	[116] = {
		["type"] = 1,
		["exp"] = 13000000000,
		["ability"] = 250,
		["lv"] = 116,
		["name_t"] = "Jumppower",
	},
}