return {
	[1] = {
		["type"] = 1,
		["exp"] = 100,
		["ability"] = 20,
		["lv"] = 1,
		["name_t"] = "Jumppower",
	},
	[2] = {
		["type"] = 1,
		["exp"] = 200,
		["ability"] = 25,
		["lv"] = 2,
		["name_t"] = "Jumppower",
	},
	[3] = {
		["type"] = 1,
		["exp"] = 300,
		["ability"] = 30,
		["lv"] = 3,
		["name_t"] = "Jumppower",
	},
	[4] = {
		["type"] = 1,
		["exp"] = 400,
		["ability"] = 35,
		["lv"] = 4,
		["name_t"] = "Jumppower",
	},
	[5] = {
		["type"] = 1,
		["exp"] = 500,
		["ability"] = 40,
		["lv"] = 5,
		["name_t"] = "Jumppower",
	},
	[6] = {
		["type"] = 1,
		["exp"] = 600,
		["ability"] = 45,
		["lv"] = 6,
		["name_t"] = "Jumppower",
	},
	[7] = {
		["type"] = 1,
		["exp"] = 700,
		["ability"] = 50,
		["lv"] = 7,
		["name_t"] = "Jumppower",
	},
	[8] = {
		["type"] = 1,
		["exp"] = 800,
		["ability"] = 55,
		["lv"] = 8,
		["name_t"] = "Jumppower",
	},
	[9] = {
		["type"] = 1,
		["exp"] = 900,
		["ability"] = 60,
		["lv"] = 9,
		["name_t"] = "Jumppower",
	},
	[10] = {
		["type"] = 1,
		["exp"] = 1000,
		["ability"] = 65,
		["lv"] = 10,
		["name_t"] = "Jumppower",
	},
	[11] = {
		["type"] = 1,
		["exp"] = 1100,
		["ability"] = 70,
		["lv"] = 6,
		["name_t"] = "Jumppower",
	},
	[12] = {
		["type"] = 1,
		["exp"] = 1200,
		["ability"] = 75,
		["lv"] = 7,
		["name_t"] = "Jumppower",
	},
	[13] = {
		["type"] = 1,
		["exp"] = 1300,
		["ability"] = 80,
		["lv"] = 8,
		["name_t"] = "Jumppower",
	},
	[14] = {
		["type"] = 1,
		["exp"] = 1400,
		["ability"] = 85,
		["lv"] = 9,
		["name_t"] = "Jumppower",
	},
	[15] = {
		["type"] = 1,
		["exp"] = 1500,
		["ability"] = 90,
		["lv"] = 10,
		["name_t"] = "Jumppower",
	},
	[16] = {
		["type"] = 1,
		["exp"] = 1600,
		["ability"] = 95,
		["lv"] = 11,
		["name_t"] = "Jumppower",
	},
	[17] = {
		["type"] = 1,
		["exp"] = 1700,
		["ability"] = 100,
		["lv"] = 12,
		["name_t"] = "Jumppower",
	},
}
