return {
	[1] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill1",
		["id"] = 1,
		["iconImageId"] = "rbxassetid://109696861846778",
		["name_t"] = "Treadmill1",
		["Column14"] = 0,
		["costCoin"] = 0,
		["productId"] = 0,
		["costRb"] = 0,
		["ability"] = 1,
		["costType"] = 1,
		["quality"] = 1,
	},
	[2] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike1",
		["id"] = 2,
		["iconImageId"] = "rbxassetid://89697596614593",
		["name_t"] = "Bike1",
		["Column14"] = 0,
		["costCoin"] = 7500,
		["productId"] = 3583247572,
		["costRb"] = 9,
		["ability"] = 3,
		["costType"] = 3,
		["quality"] = 1,
	},
	[3] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill2",
		["id"] = 3,
		["iconImageId"] = "rbxassetid://79948992738066",
		["name_t"] = "Treadmill2",
		["Column14"] = 0,
		["costCoin"] = 75000,
		["productId"] = 0,
		["costRb"] = 25,
		["ability"] = 6,
		["costType"] = 3,
		["quality"] = 2,
	},
	[4] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike2",
		["id"] = 4,
		["iconImageId"] = "rbxassetid://140367021331621",
		["name_t"] = "Bike2",
		["Column14"] = 0,
		["costCoin"] = 1200000,
		["productId"] = 3583247628,
		["costRb"] = 80,
		["ability"] = 16,
		["costType"] = 3,
		["quality"] = 2,
	},
	[5] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill3",
		["id"] = 5,
		["iconImageId"] = "rbxassetid://81633780206590",
		["name_t"] = "Treadmill3",
		["Column14"] = 0,
		["costCoin"] = 18000000,
		["productId"] = 0,
		["costRb"] = 160,
		["ability"] = 40,
		["costType"] = 3,
		["quality"] = 3,
	},
	[6] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike3",
		["id"] = 6,
		["iconImageId"] = "rbxassetid://75638913691987",
		["name_t"] = "Bike3",
		["Column14"] = 0,
		["costCoin"] = 360000000,
		["productId"] = 3583247683,
		["costRb"] = 280,
		["ability"] = 100,
		["costType"] = 3,
		["quality"] = 3,
	},
	[7] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill4",
		["id"] = 7,
		["iconImageId"] = "rbxassetid://125285661707160",
		["name_t"] = "Treadmill4",
		["Column14"] = 0,
		["costCoin"] = 6500000000,
		["productId"] = 0,
		["costRb"] = 400,
		["ability"] = 250,
		["costType"] = 3,
		["quality"] = 4,
	},
	[8] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike4",
		["id"] = 8,
		["iconImageId"] = "rbxassetid://126256460849747",
		["name_t"] = "Bike4",
		["Column14"] = 0,
		["costCoin"] = 120000000000,
		["productId"] = 0,
		["costRb"] = 480,
		["ability"] = 600,
		["costType"] = 3,
		["quality"] = 4,
	},
	[9] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill5",
		["id"] = 9,
		["iconImageId"] = "rbxassetid://83213072882621",
		["name_t"] = "Treadmill5",
		["Column14"] = 0,
		["costCoin"] = 2100000000000,
		["productId"] = 0,
		["costRb"] = 636,
		["ability"] = 1500,
		["costType"] = 3,
		["quality"] = 5,
	},
	[10] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike5",
		["id"] = 10,
		["iconImageId"] = "rbxassetid://88450114804805",
		["name_t"] = "Bike5",
		["Column14"] = 0,
		["costCoin"] = 38000000000000,
		["productId"] = 0,
		["costRb"] = 720,
		["ability"] = 3800,
		["costType"] = 3,
		["quality"] = 5,
	},
	[11] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill6",
		["id"] = 11,
		["iconImageId"] = "rbxassetid://70834741089669",
		["name_t"] = "Treadmill6",
		["Column14"] = 0,
		["costCoin"] = 680000000000000,
		["productId"] = 0,
		["costRb"] = 960,
		["ability"] = 9600,
		["costType"] = 3,
		["quality"] = 6,
	},
	[12] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike6",
		["id"] = 12,
		["iconImageId"] = "rbxassetid://101496326491409",
		["name_t"] = "Bike6",
		["Column14"] = 0,
		["costCoin"] = 12000000000000000,
		["productId"] = 0,
		["costRb"] = 1160,
		["ability"] = 24000,
		["costType"] = 3,
		["quality"] = 6,
	},
	[13] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill7",
		["id"] = 13,
		["iconImageId"] = "rbxassetid://87156152714446",
		["name_t"] = "Treadmill7",
		["Column14"] = 0,
		["costCoin"] = 220000000000000000,
		["productId"] = 0,
		["costRb"] = 1600,
		["ability"] = 60000,
		["costType"] = 3,
		["quality"] = 7,
	},
	[14] = {
		["Column13"] = 0,
		["type"] = 2,
		["resourceName"] = "Bike7",
		["id"] = 14,
		["iconImageId"] = "rbxassetid://117736147046687",
		["name_t"] = "Bike7",
		["Column14"] = 0,
		["costCoin"] = 4000000000000000000,
		["productId"] = 0,
		["costRb"] = 2480,
		["ability"] = 150000,
		["costType"] = 3,
		["quality"] = 7,
	},
	[15] = {
		["Column13"] = 0,
		["type"] = 1,
		["resourceName"] = "Treadmill8",
		["id"] = 15,
		["iconImageId"] = "rbxassetid://124534541831922",
		["name_t"] = "Treadmill8",
		["Column14"] = 0,
		["costCoin"] = 71000000000000000000,
		["productId"] = 0,
		["costRb"] = 3280,
		["ability"] = 380000,
		["costType"] = 3,
		["quality"] = 8,
	},
}