return {
	[1] = {
		["type"] = 1,
		["exp"] = 5,
		["ability"] = 16,
		["lv"] = 1,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[2] = {
		["type"] = 1,
		["exp"] = 10,
		["ability"] = 18,
		["lv"] = 2,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[3] = {
		["type"] = 1,
		["exp"] = 25,
		["ability"] = 20,
		["lv"] = 3,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[4] = {
		["type"] = 1,
		["exp"] = 40,
		["ability"] = 22,
		["lv"] = 4,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[5] = {
		["type"] = 1,
		["exp"] = 60,
		["ability"] = 23,
		["lv"] = 5,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[6] = {
		["type"] = 1,
		["exp"] = 72,
		["ability"] = 24,
		["lv"] = 6,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[7] = {
		["type"] = 1,
		["exp"] = 86,
		["ability"] = 25,
		["lv"] = 7,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[8] = {
		["type"] = 1,
		["exp"] = 100,
		["ability"] = 26,
		["lv"] = 8,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 31,
	},
	[9] = {
		["type"] = 1,
		["exp"] = 120,
		["ability"] = 27,
		["lv"] = 9,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 31,
	},
	[10] = {
		["type"] = 1,
		["exp"] = 270,
		["ability"] = 28,
		["lv"] = 10,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 31,
	},
	[11] = {
		["type"] = 1,
		["exp"] = 150,
		["ability"] = 29,
		["lv"] = 11,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 32,
	},
	[12] = {
		["type"] = 1,
		["exp"] = 180,
		["ability"] = 30,
		["lv"] = 12,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 32,
	},
	[13] = {
		["type"] = 1,
		["exp"] = 220,
		["ability"] = 31,
		["lv"] = 13,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 33,
	},
	[14] = {
		["type"] = 1,
		["exp"] = 260,
		["ability"] = 32,
		["lv"] = 14,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 33,
	},
	[15] = {
		["type"] = 1,
		["exp"] = 480,
		["ability"] = 33,
		["lv"] = 15,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 34,
	},
	[16] = {
		["type"] = 1,
		["exp"] = 380,
		["ability"] = 34,
		["lv"] = 16,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 34,
	},
	[17] = {
		["type"] = 1,
		["exp"] = 450,
		["ability"] = 35,
		["lv"] = 17,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 34,
	},
	[18] = {
		["type"] = 1,
		["exp"] = 540,
		["ability"] = 36,
		["lv"] = 18,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 35,
	},
	[19] = {
		["type"] = 1,
		["exp"] = 650,
		["ability"] = 37,
		["lv"] = 19,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 35,
	},
	[20] = {
		["type"] = 1,
		["exp"] = 1700,
		["ability"] = 38,
		["lv"] = 20,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 36,
	},
	[21] = {
		["type"] = 1,
		["exp"] = 860,
		["ability"] = 39,
		["lv"] = 21,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 36,
	},
	[22] = {
		["type"] = 1,
		["exp"] = 950,
		["ability"] = 40,
		["lv"] = 22,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 37,
	},
	[23] = {
		["type"] = 1,
		["exp"] = 1000,
		["ability"] = 41,
		["lv"] = 23,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 37,
	},
	[24] = {
		["type"] = 1,
		["exp"] = 1100,
		["ability"] = 42,
		["lv"] = 24,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 38,
	},
	[25] = {
		["type"] = 1,
		["exp"] = 1900,
		["ability"] = 43,
		["lv"] = 25,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 38,
	},
	[26] = {
		["type"] = 1,
		["exp"] = 1400,
		["ability"] = 44,
		["lv"] = 26,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 39,
	},
	[27] = {
		["type"] = 1,
		["exp"] = 1500,
		["ability"] = 45,
		["lv"] = 27,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 41,
	},
	[28] = {
		["type"] = 1,
		["exp"] = 1700,
		["ability"] = 46,
		["lv"] = 28,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 43,
	},
	[29] = {
		["type"] = 1,
		["exp"] = 1800,
		["ability"] = 47,
		["lv"] = 29,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 44,
	},
	[30] = {
		["type"] = 1,
		["exp"] = 4300,
		["ability"] = 48,
		["lv"] = 30,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 46,
	},
	[31] = {
		["type"] = 1,
		["exp"] = 2200,
		["ability"] = 49,
		["lv"] = 31,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 48,
	},
	[32] = {
		["type"] = 1,
		["exp"] = 2500,
		["ability"] = 50,
		["lv"] = 32,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 50,
	},
	[33] = {
		["type"] = 1,
		["exp"] = 2700,
		["ability"] = 51,
		["lv"] = 33,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 52,
	},
	[34] = {
		["type"] = 1,
		["exp"] = 3000,
		["ability"] = 52,
		["lv"] = 34,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 54,
	},
	[35] = {
		["type"] = 1,
		["exp"] = 5000,
		["ability"] = 53,
		["lv"] = 35,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 57,
	},
	[36] = {
		["type"] = 1,
		["exp"] = 3600,
		["ability"] = 54,
		["lv"] = 36,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 59,
	},
	[37] = {
		["type"] = 1,
		["exp"] = 3900,
		["ability"] = 55,
		["lv"] = 37,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 61,
	},
	[38] = {
		["type"] = 1,
		["exp"] = 4300,
		["ability"] = 56,
		["lv"] = 38,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 63,
	},
	[39] = {
		["type"] = 1,
		["exp"] = 4800,
		["ability"] = 57,
		["lv"] = 39,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 65,
	},
	[40] = {
		["type"] = 1,
		["exp"] = 11000,
		["ability"] = 58,
		["lv"] = 40,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 68,
	},
	[41] = {
		["type"] = 1,
		["exp"] = 5800,
		["ability"] = 59,
		["lv"] = 41,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 70,
	},
	[42] = {
		["type"] = 1,
		["exp"] = 6400,
		["ability"] = 60,
		["lv"] = 42,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 72,
	},
	[43] = {
		["type"] = 1,
		["exp"] = 7000,
		["ability"] = 61,
		["lv"] = 43,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 75,
	},
	[44] = {
		["type"] = 1,
		["exp"] = 7700,
		["ability"] = 62,
		["lv"] = 44,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 77,
	},
	[45] = {
		["type"] = 1,
		["exp"] = 13000,
		["ability"] = 63,
		["lv"] = 45,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 80,
	},
	[46] = {
		["type"] = 1,
		["exp"] = 9300,
		["ability"] = 64,
		["lv"] = 46,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 82,
	},
	[47] = {
		["type"] = 1,
		["exp"] = 10000,
		["ability"] = 65,
		["lv"] = 47,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 85,
	},
	[48] = {
		["type"] = 1,
		["exp"] = 11000,
		["ability"] = 66,
		["lv"] = 48,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 87,
	},
	[49] = {
		["type"] = 1,
		["exp"] = 12000,
		["ability"] = 67,
		["lv"] = 49,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 90,
	},
	[50] = {
		["type"] = 1,
		["exp"] = 29000,
		["ability"] = 68,
		["lv"] = 50,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 93,
	},
	[51] = {
		["type"] = 1,
		["exp"] = 15000,
		["ability"] = 69,
		["lv"] = 51,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 95,
	},
	[52] = {
		["type"] = 1,
		["exp"] = 16000,
		["ability"] = 70,
		["lv"] = 52,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 98,
	},
	[53] = {
		["type"] = 1,
		["exp"] = 18000,
		["ability"] = 71,
		["lv"] = 53,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 101,
	},
	[54] = {
		["type"] = 1,
		["exp"] = 20000,
		["ability"] = 72,
		["lv"] = 54,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 104,
	},
	[55] = {
		["type"] = 1,
		["exp"] = 34000,
		["ability"] = 73,
		["lv"] = 55,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 106,
	},
	[56] = {
		["type"] = 1,
		["exp"] = 24000,
		["ability"] = 74,
		["lv"] = 56,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 109,
	},
	[57] = {
		["type"] = 1,
		["exp"] = 27000,
		["ability"] = 75,
		["lv"] = 57,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 112,
	},
	[58] = {
		["type"] = 1,
		["exp"] = 29000,
		["ability"] = 76,
		["lv"] = 58,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 115,
	},
	[59] = {
		["type"] = 1,
		["exp"] = 32000,
		["ability"] = 77,
		["lv"] = 59,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 118,
	},
	[60] = {
		["type"] = 1,
		["exp"] = 75000,
		["ability"] = 78,
		["lv"] = 60,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 121,
	},
	[61] = {
		["type"] = 1,
		["exp"] = 39000,
		["ability"] = 79,
		["lv"] = 61,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 124,
	},
	[62] = {
		["type"] = 1,
		["exp"] = 43000,
		["ability"] = 80,
		["lv"] = 62,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 127,
	},
	[63] = {
		["type"] = 1,
		["exp"] = 47000,
		["ability"] = 81,
		["lv"] = 63,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 130,
	},
	[64] = {
		["type"] = 1,
		["exp"] = 52000,
		["ability"] = 82,
		["lv"] = 64,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 134,
	},
	[65] = {
		["type"] = 1,
		["exp"] = 88000,
		["ability"] = 83,
		["lv"] = 65,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 137,
	},
	[66] = {
		["type"] = 1,
		["exp"] = 63000,
		["ability"] = 84,
		["lv"] = 66,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 140,
	},
	[67] = {
		["type"] = 1,
		["exp"] = 69000,
		["ability"] = 85,
		["lv"] = 67,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 143,
	},
	[68] = {
		["type"] = 1,
		["exp"] = 76000,
		["ability"] = 86,
		["lv"] = 68,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 147,
	},
	[69] = {
		["type"] = 1,
		["exp"] = 83000,
		["ability"] = 87,
		["lv"] = 69,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 150,
	},
	[70] = {
		["type"] = 1,
		["exp"] = 200000,
		["ability"] = 88,
		["lv"] = 70,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 153,
	},
	[71] = {
		["type"] = 1,
		["exp"] = 100000,
		["ability"] = 89,
		["lv"] = 71,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 157,
	},
	[72] = {
		["type"] = 1,
		["exp"] = 110000,
		["ability"] = 90,
		["lv"] = 72,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 160,
	},
	[73] = {
		["type"] = 1,
		["exp"] = 120000,
		["ability"] = 91,
		["lv"] = 73,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 164,
	},
	[74] = {
		["type"] = 1,
		["exp"] = 130000,
		["ability"] = 92,
		["lv"] = 74,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 167,
	},
	[75] = {
		["type"] = 1,
		["exp"] = 230000,
		["ability"] = 93,
		["lv"] = 75,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 171,
	},
	[76] = {
		["type"] = 1,
		["exp"] = 160000,
		["ability"] = 94,
		["lv"] = 76,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 174,
	},
	[77] = {
		["type"] = 1,
		["exp"] = 180000,
		["ability"] = 95,
		["lv"] = 77,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 178,
	},
	[78] = {
		["type"] = 1,
		["exp"] = 200000,
		["ability"] = 96,
		["lv"] = 78,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 181,
	},
	[79] = {
		["type"] = 1,
		["exp"] = 220000,
		["ability"] = 97,
		["lv"] = 79,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 185,
	},
	[80] = {
		["type"] = 1,
		["exp"] = 510000,
		["ability"] = 98,
		["lv"] = 80,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 189,
	},
	[81] = {
		["type"] = 1,
		["exp"] = 260000,
		["ability"] = 99,
		["lv"] = 81,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 193,
	},
	[82] = {
		["type"] = 1,
		["exp"] = 290000,
		["ability"] = 100,
		["lv"] = 82,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 196,
	},
	[83] = {
		["type"] = 1,
		["exp"] = 320000,
		["ability"] = 101,
		["lv"] = 83,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 200,
	},
	[84] = {
		["type"] = 1,
		["exp"] = 350000,
		["ability"] = 102,
		["lv"] = 84,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 204,
	},
	[85] = {
		["type"] = 1,
		["exp"] = 590000,
		["ability"] = 103,
		["lv"] = 85,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 208,
	},
	[86] = {
		["type"] = 1,
		["exp"] = 420000,
		["ability"] = 104,
		["lv"] = 86,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 212,
	},
	[87] = {
		["type"] = 1,
		["exp"] = 460000,
		["ability"] = 105,
		["lv"] = 87,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 216,
	},
	[88] = {
		["type"] = 1,
		["exp"] = 510000,
		["ability"] = 106,
		["lv"] = 88,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 220,
	},
	[89] = {
		["type"] = 1,
		["exp"] = 560000,
		["ability"] = 107,
		["lv"] = 89,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 224,
	},
	[90] = {
		["type"] = 1,
		["exp"] = 1300000,
		["ability"] = 108,
		["lv"] = 90,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 228,
	},
	[91] = {
		["type"] = 1,
		["exp"] = 680000,
		["ability"] = 109,
		["lv"] = 91,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 232,
	},
	[92] = {
		["type"] = 1,
		["exp"] = 750000,
		["ability"] = 110,
		["lv"] = 92,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 236,
	},
	[93] = {
		["type"] = 1,
		["exp"] = 820000,
		["ability"] = 111,
		["lv"] = 93,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 241,
	},
	[94] = {
		["type"] = 1,
		["exp"] = 900000,
		["ability"] = 112,
		["lv"] = 94,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 245,
	},
	[95] = {
		["type"] = 1,
		["exp"] = 1500000,
		["ability"] = 113,
		["lv"] = 95,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 249,
	},
	[96] = {
		["type"] = 1,
		["exp"] = 1100000,
		["ability"] = 114,
		["lv"] = 96,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 253,
	},
	[97] = {
		["type"] = 1,
		["exp"] = 1200000,
		["ability"] = 115,
		["lv"] = 97,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 258,
	},
	[98] = {
		["type"] = 1,
		["exp"] = 1300000,
		["ability"] = 116,
		["lv"] = 98,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 262,
	},
	[99] = {
		["type"] = 1,
		["exp"] = 1500000,
		["ability"] = 117,
		["lv"] = 99,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 267,
	},
	[100] = {
		["type"] = 1,
		["exp"] = 3400000,
		["ability"] = 118,
		["lv"] = 100,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 271,
	},
	[101] = {
		["type"] = 1,
		["exp"] = 1800000,
		["ability"] = 119,
		["lv"] = 101,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 275,
	},
	[102] = {
		["type"] = 1,
		["exp"] = 1900000,
		["ability"] = 120,
		["lv"] = 102,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 280,
	},
	[103] = {
		["type"] = 1,
		["exp"] = 2100000,
		["ability"] = 121,
		["lv"] = 103,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 284,
	},
	[104] = {
		["type"] = 1,
		["exp"] = 2300000,
		["ability"] = 122,
		["lv"] = 104,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 289,
	},
	[105] = {
		["type"] = 1,
		["exp"] = 4000000,
		["ability"] = 123,
		["lv"] = 105,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 294,
	},
	[106] = {
		["type"] = 1,
		["exp"] = 2800000,
		["ability"] = 124,
		["lv"] = 106,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 298,
	},
	[107] = {
		["type"] = 1,
		["exp"] = 3100000,
		["ability"] = 125,
		["lv"] = 107,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[108] = {
		["type"] = 1,
		["exp"] = 3400000,
		["ability"] = 126,
		["lv"] = 108,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[109] = {
		["type"] = 1,
		["exp"] = 3800000,
		["ability"] = 127,
		["lv"] = 109,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[110] = {
		["type"] = 1,
		["exp"] = 8800000,
		["ability"] = 128,
		["lv"] = 110,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[111] = {
		["type"] = 1,
		["exp"] = 4600000,
		["ability"] = 129,
		["lv"] = 111,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[112] = {
		["type"] = 1,
		["exp"] = 5000000,
		["ability"] = 130,
		["lv"] = 112,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[113] = {
		["type"] = 1,
		["exp"] = 5500000,
		["ability"] = 131,
		["lv"] = 113,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[114] = {
		["type"] = 1,
		["exp"] = 6100000,
		["ability"] = 132,
		["lv"] = 114,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[115] = {
		["type"] = 1,
		["exp"] = 10000000,
		["ability"] = 133,
		["lv"] = 115,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[116] = {
		["type"] = 1,
		["exp"] = 7400000,
		["ability"] = 134,
		["lv"] = 116,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[117] = {
		["type"] = 1,
		["exp"] = 8100000,
		["ability"] = 135,
		["lv"] = 117,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[118] = {
		["type"] = 1,
		["exp"] = 8900000,
		["ability"] = 136,
		["lv"] = 118,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[119] = {
		["type"] = 1,
		["exp"] = 9800000,
		["ability"] = 137,
		["lv"] = 119,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[120] = {
		["type"] = 1,
		["exp"] = 23000000,
		["ability"] = 138,
		["lv"] = 120,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[121] = {
		["type"] = 1,
		["exp"] = 12000000,
		["ability"] = 139,
		["lv"] = 121,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[122] = {
		["type"] = 1,
		["exp"] = 13000000,
		["ability"] = 140,
		["lv"] = 122,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[123] = {
		["type"] = 1,
		["exp"] = 14000000,
		["ability"] = 141,
		["lv"] = 123,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[124] = {
		["type"] = 1,
		["exp"] = 16000000,
		["ability"] = 142,
		["lv"] = 124,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[125] = {
		["type"] = 1,
		["exp"] = 27000000,
		["ability"] = 143,
		["lv"] = 125,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[126] = {
		["type"] = 1,
		["exp"] = 19000000,
		["ability"] = 144,
		["lv"] = 126,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[127] = {
		["type"] = 1,
		["exp"] = 21000000,
		["ability"] = 145,
		["lv"] = 127,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[128] = {
		["type"] = 1,
		["exp"] = 23000000,
		["ability"] = 146,
		["lv"] = 128,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[129] = {
		["type"] = 1,
		["exp"] = 25000000,
		["ability"] = 147,
		["lv"] = 129,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[130] = {
		["type"] = 1,
		["exp"] = 59000000,
		["ability"] = 148,
		["lv"] = 130,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[131] = {
		["type"] = 1,
		["exp"] = 31000000,
		["ability"] = 149,
		["lv"] = 131,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[132] = {
		["type"] = 1,
		["exp"] = 34000000,
		["ability"] = 150,
		["lv"] = 132,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[133] = {
		["type"] = 1,
		["exp"] = 37000000,
		["ability"] = 151,
		["lv"] = 133,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[134] = {
		["type"] = 1,
		["exp"] = 41000000,
		["ability"] = 152,
		["lv"] = 134,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[135] = {
		["type"] = 1,
		["exp"] = 69000000,
		["ability"] = 153,
		["lv"] = 135,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[136] = {
		["type"] = 0,
		["exp"] = 49000000,
		["ability"] = 154,
		["lv"] = 136,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[137] = {
		["type"] = 0,
		["exp"] = 54000000,
		["ability"] = 155,
		["lv"] = 137,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[138] = {
		["type"] = 0,
		["exp"] = 60000000,
		["ability"] = 156,
		["lv"] = 138,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[139] = {
		["type"] = 0,
		["exp"] = 66000000,
		["ability"] = 157,
		["lv"] = 139,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[140] = {
		["type"] = 0,
		["exp"] = 150000000,
		["ability"] = 158,
		["lv"] = 140,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[141] = {
		["type"] = 0,
		["exp"] = 80000000,
		["ability"] = 159,
		["lv"] = 141,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[142] = {
		["type"] = 0,
		["exp"] = 88000000,
		["ability"] = 160,
		["lv"] = 142,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[143] = {
		["type"] = 0,
		["exp"] = 96000000,
		["ability"] = 161,
		["lv"] = 143,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[144] = {
		["type"] = 0,
		["exp"] = 110000000,
		["ability"] = 162,
		["lv"] = 144,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[145] = {
		["type"] = 0,
		["exp"] = 180000000,
		["ability"] = 163,
		["lv"] = 145,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[146] = {
		["type"] = 0,
		["exp"] = 130000000,
		["ability"] = 164,
		["lv"] = 146,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[147] = {
		["type"] = 0,
		["exp"] = 140000000,
		["ability"] = 165,
		["lv"] = 147,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[148] = {
		["type"] = 0,
		["exp"] = 160000000,
		["ability"] = 166,
		["lv"] = 148,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[149] = {
		["type"] = 0,
		["exp"] = 170000000,
		["ability"] = 167,
		["lv"] = 149,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[150] = {
		["type"] = 0,
		["exp"] = 400000000,
		["ability"] = 168,
		["lv"] = 150,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[151] = {
		["type"] = 0,
		["exp"] = 210000000,
		["ability"] = 169,
		["lv"] = 151,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[152] = {
		["type"] = 0,
		["exp"] = 230000000,
		["ability"] = 170,
		["lv"] = 152,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[153] = {
		["type"] = 0,
		["exp"] = 250000000,
		["ability"] = 171,
		["lv"] = 153,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[154] = {
		["type"] = 0,
		["exp"] = 280000000,
		["ability"] = 172,
		["lv"] = 154,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[155] = {
		["type"] = 0,
		["exp"] = 470000000,
		["ability"] = 173,
		["lv"] = 155,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[156] = {
		["type"] = 0,
		["exp"] = 330000000,
		["ability"] = 174,
		["lv"] = 156,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[157] = {
		["type"] = 0,
		["exp"] = 370000000,
		["ability"] = 175,
		["lv"] = 157,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[158] = {
		["type"] = 0,
		["exp"] = 400000000,
		["ability"] = 176,
		["lv"] = 158,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[159] = {
		["type"] = 0,
		["exp"] = 440000000,
		["ability"] = 177,
		["lv"] = 159,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[160] = {
		["type"] = 0,
		["exp"] = 1000000000,
		["ability"] = 178,
		["lv"] = 160,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[161] = {
		["type"] = 0,
		["exp"] = 540000000,
		["ability"] = 179,
		["lv"] = 161,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[162] = {
		["type"] = 0,
		["exp"] = 590000000,
		["ability"] = 180,
		["lv"] = 162,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[163] = {
		["type"] = 0,
		["exp"] = 650000000,
		["ability"] = 181,
		["lv"] = 163,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[164] = {
		["type"] = 0,
		["exp"] = 710000000,
		["ability"] = 182,
		["lv"] = 164,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[165] = {
		["type"] = 0,
		["exp"] = 1200000000,
		["ability"] = 183,
		["lv"] = 165,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[166] = {
		["type"] = 0,
		["exp"] = 860000000,
		["ability"] = 184,
		["lv"] = 166,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[167] = {
		["type"] = 0,
		["exp"] = 950000000,
		["ability"] = 185,
		["lv"] = 167,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[168] = {
		["type"] = 0,
		["exp"] = 1000000000,
		["ability"] = 186,
		["lv"] = 168,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[169] = {
		["type"] = 0,
		["exp"] = 1100000000,
		["ability"] = 187,
		["lv"] = 169,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[170] = {
		["type"] = 0,
		["exp"] = 2700000000,
		["ability"] = 188,
		["lv"] = 170,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[171] = {
		["type"] = 0,
		["exp"] = 1400000000,
		["ability"] = 189,
		["lv"] = 171,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[172] = {
		["type"] = 0,
		["exp"] = 1500000000,
		["ability"] = 190,
		["lv"] = 172,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[173] = {
		["type"] = 0,
		["exp"] = 1700000000,
		["ability"] = 191,
		["lv"] = 173,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[174] = {
		["type"] = 0,
		["exp"] = 1900000000,
		["ability"] = 192,
		["lv"] = 174,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[175] = {
		["type"] = 0,
		["exp"] = 3100000000,
		["ability"] = 193,
		["lv"] = 175,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[176] = {
		["type"] = 0,
		["exp"] = 2200000000,
		["ability"] = 194,
		["lv"] = 176,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[177] = {
		["type"] = 0,
		["exp"] = 2500000000,
		["ability"] = 195,
		["lv"] = 177,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[178] = {
		["type"] = 0,
		["exp"] = 2700000000,
		["ability"] = 196,
		["lv"] = 178,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[179] = {
		["type"] = 0,
		["exp"] = 3000000000,
		["ability"] = 197,
		["lv"] = 179,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[180] = {
		["type"] = 0,
		["exp"] = 7000000000,
		["ability"] = 198,
		["lv"] = 180,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[181] = {
		["type"] = 0,
		["exp"] = 3600000000,
		["ability"] = 199,
		["lv"] = 181,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[182] = {
		["type"] = 0,
		["exp"] = 4000000000,
		["ability"] = 200,
		["lv"] = 182,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[183] = {
		["type"] = 0,
		["exp"] = 4400000000,
		["ability"] = 201,
		["lv"] = 183,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[184] = {
		["type"] = 0,
		["exp"] = 4800000000,
		["ability"] = 202,
		["lv"] = 184,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[185] = {
		["type"] = 0,
		["exp"] = 8100000000,
		["ability"] = 203,
		["lv"] = 185,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[186] = {
		["type"] = 0,
		["exp"] = 5800000000,
		["ability"] = 204,
		["lv"] = 186,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[187] = {
		["type"] = 0,
		["exp"] = 6400000000,
		["ability"] = 205,
		["lv"] = 187,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[188] = {
		["type"] = 0,
		["exp"] = 7000000000,
		["ability"] = 206,
		["lv"] = 188,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[189] = {
		["type"] = 0,
		["exp"] = 7700000000,
		["ability"] = 207,
		["lv"] = 189,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[190] = {
		["type"] = 0,
		["exp"] = 18000000000,
		["ability"] = 208,
		["lv"] = 190,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[191] = {
		["type"] = 0,
		["exp"] = 9400000000,
		["ability"] = 209,
		["lv"] = 191,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[192] = {
		["type"] = 0,
		["exp"] = 10000000000,
		["ability"] = 210,
		["lv"] = 192,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[193] = {
		["type"] = 0,
		["exp"] = 11000000000,
		["ability"] = 211,
		["lv"] = 193,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[194] = {
		["type"] = 0,
		["exp"] = 12000000000,
		["ability"] = 212,
		["lv"] = 194,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[195] = {
		["type"] = 0,
		["exp"] = 21000000000,
		["ability"] = 213,
		["lv"] = 195,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[196] = {
		["type"] = 0,
		["exp"] = 15000000000,
		["ability"] = 214,
		["lv"] = 196,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[197] = {
		["type"] = 0,
		["exp"] = 17000000000,
		["ability"] = 215,
		["lv"] = 197,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[198] = {
		["type"] = 0,
		["exp"] = 18000000000,
		["ability"] = 216,
		["lv"] = 198,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[199] = {
		["type"] = 0,
		["exp"] = 20000000000,
		["ability"] = 217,
		["lv"] = 199,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[200] = {
		["type"] = 0,
		["exp"] = 47000000000,
		["ability"] = 218,
		["lv"] = 200,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[201] = {
		["type"] = 0,
		["exp"] = 24000000000,
		["ability"] = 219,
		["lv"] = 201,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[202] = {
		["type"] = 0,
		["exp"] = 27000000000,
		["ability"] = 220,
		["lv"] = 202,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[203] = {
		["type"] = 0,
		["exp"] = 29000000000,
		["ability"] = 221,
		["lv"] = 203,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[204] = {
		["type"] = 0,
		["exp"] = 32000000000,
		["ability"] = 222,
		["lv"] = 204,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[205] = {
		["type"] = 0,
		["exp"] = 55000000000,
		["ability"] = 223,
		["lv"] = 205,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[206] = {
		["type"] = 0,
		["exp"] = 39000000000,
		["ability"] = 224,
		["lv"] = 206,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[207] = {
		["type"] = 0,
		["exp"] = 43000000000,
		["ability"] = 225,
		["lv"] = 207,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[208] = {
		["type"] = 0,
		["exp"] = 47000000000,
		["ability"] = 226,
		["lv"] = 208,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[209] = {
		["type"] = 0,
		["exp"] = 52000000000,
		["ability"] = 227,
		["lv"] = 209,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[210] = {
		["type"] = 0,
		["exp"] = 120000000000,
		["ability"] = 228,
		["lv"] = 210,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[211] = {
		["type"] = 0,
		["exp"] = 63000000000,
		["ability"] = 229,
		["lv"] = 211,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[212] = {
		["type"] = 0,
		["exp"] = 69000000000,
		["ability"] = 230,
		["lv"] = 212,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[213] = {
		["type"] = 0,
		["exp"] = 76000000000,
		["ability"] = 231,
		["lv"] = 213,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[214] = {
		["type"] = 0,
		["exp"] = 84000000000,
		["ability"] = 232,
		["lv"] = 214,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[215] = {
		["type"] = 0,
		["exp"] = 140000000000,
		["ability"] = 233,
		["lv"] = 215,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[216] = {
		["type"] = 0,
		["exp"] = 100000000000,
		["ability"] = 234,
		["lv"] = 216,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[217] = {
		["type"] = 0,
		["exp"] = 110000000000,
		["ability"] = 235,
		["lv"] = 217,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[218] = {
		["type"] = 0,
		["exp"] = 120000000000,
		["ability"] = 236,
		["lv"] = 218,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[219] = {
		["type"] = 0,
		["exp"] = 130000000000,
		["ability"] = 237,
		["lv"] = 219,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[220] = {
		["type"] = 0,
		["exp"] = 320000000000,
		["ability"] = 238,
		["lv"] = 220,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[221] = {
		["type"] = 0,
		["exp"] = 160000000000,
		["ability"] = 239,
		["lv"] = 221,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[222] = {
		["type"] = 0,
		["exp"] = 180000000000,
		["ability"] = 240,
		["lv"] = 222,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[223] = {
		["type"] = 0,
		["exp"] = 200000000000,
		["ability"] = 241,
		["lv"] = 223,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[224] = {
		["type"] = 0,
		["exp"] = 220000000000,
		["ability"] = 242,
		["lv"] = 224,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[225] = {
		["type"] = 0,
		["exp"] = 370000000000,
		["ability"] = 243,
		["lv"] = 225,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[226] = {
		["type"] = 0,
		["exp"] = 260000000000,
		["ability"] = 244,
		["lv"] = 226,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[227] = {
		["type"] = 0,
		["exp"] = 290000000000,
		["ability"] = 245,
		["lv"] = 227,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[228] = {
		["type"] = 0,
		["exp"] = 320000000000,
		["ability"] = 246,
		["lv"] = 228,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[229] = {
		["type"] = 0,
		["exp"] = 350000000000,
		["ability"] = 247,
		["lv"] = 229,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[230] = {
		["type"] = 0,
		["exp"] = 820000000000,
		["ability"] = 248,
		["lv"] = 230,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[231] = {
		["type"] = 0,
		["exp"] = 420000000000,
		["ability"] = 249,
		["lv"] = 231,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[232] = {
		["type"] = 0,
		["exp"] = 470000000000,
		["ability"] = 250,
		["lv"] = 232,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[233] = {
		["type"] = 0,
		["exp"] = 510000000000,
		["ability"] = 251,
		["lv"] = 233,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[234] = {
		["type"] = 0,
		["exp"] = 560000000000,
		["ability"] = 252,
		["lv"] = 234,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[235] = {
		["type"] = 0,
		["exp"] = 950000000000,
		["ability"] = 253,
		["lv"] = 235,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[236] = {
		["type"] = 0,
		["exp"] = 680000000000,
		["ability"] = 254,
		["lv"] = 236,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[237] = {
		["type"] = 0,
		["exp"] = 750000000000,
		["ability"] = 255,
		["lv"] = 237,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[238] = {
		["type"] = 0,
		["exp"] = 820000000000,
		["ability"] = 256,
		["lv"] = 238,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[239] = {
		["type"] = 0,
		["exp"] = 910000000000,
		["ability"] = 257,
		["lv"] = 239,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[240] = {
		["type"] = 0,
		["exp"] = 2100000000000,
		["ability"] = 258,
		["lv"] = 240,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[241] = {
		["type"] = 0,
		["exp"] = 1100000000000,
		["ability"] = 259,
		["lv"] = 241,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[242] = {
		["type"] = 0,
		["exp"] = 1200000000000,
		["ability"] = 260,
		["lv"] = 242,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[243] = {
		["type"] = 0,
		["exp"] = 1300000000000,
		["ability"] = 261,
		["lv"] = 243,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[244] = {
		["type"] = 0,
		["exp"] = 1500000000000,
		["ability"] = 262,
		["lv"] = 244,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[245] = {
		["type"] = 0,
		["exp"] = 2500000000000,
		["ability"] = 263,
		["lv"] = 245,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[246] = {
		["type"] = 0,
		["exp"] = 1800000000000,
		["ability"] = 264,
		["lv"] = 246,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[247] = {
		["type"] = 0,
		["exp"] = 1900000000000,
		["ability"] = 265,
		["lv"] = 247,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[248] = {
		["type"] = 0,
		["exp"] = 2100000000000,
		["ability"] = 266,
		["lv"] = 248,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[249] = {
		["type"] = 0,
		["exp"] = 2400000000000,
		["ability"] = 267,
		["lv"] = 249,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[250] = {
		["type"] = 0,
		["exp"] = 5500000000000,
		["ability"] = 268,
		["lv"] = 250,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[251] = {
		["type"] = 0,
		["exp"] = 2800000000000,
		["ability"] = 269,
		["lv"] = 251,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[252] = {
		["type"] = 0,
		["exp"] = 3100000000000,
		["ability"] = 270,
		["lv"] = 252,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[253] = {
		["type"] = 0,
		["exp"] = 3400000000000,
		["ability"] = 271,
		["lv"] = 253,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[254] = {
		["type"] = 0,
		["exp"] = 3800000000000,
		["ability"] = 272,
		["lv"] = 254,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[255] = {
		["type"] = 0,
		["exp"] = 6400000000000,
		["ability"] = 273,
		["lv"] = 255,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[256] = {
		["type"] = 0,
		["exp"] = 4600000000000,
		["ability"] = 274,
		["lv"] = 256,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[257] = {
		["type"] = 0,
		["exp"] = 5000000000000,
		["ability"] = 275,
		["lv"] = 257,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[258] = {
		["type"] = 0,
		["exp"] = 5500000000000,
		["ability"] = 276,
		["lv"] = 258,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[259] = {
		["type"] = 0,
		["exp"] = 6100000000000,
		["ability"] = 277,
		["lv"] = 259,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[260] = {
		["type"] = 0,
		["exp"] = 14000000000000,
		["ability"] = 278,
		["lv"] = 260,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[261] = {
		["type"] = 0,
		["exp"] = 7400000000000,
		["ability"] = 279,
		["lv"] = 261,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[262] = {
		["type"] = 0,
		["exp"] = 8100000000000,
		["ability"] = 280,
		["lv"] = 262,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[263] = {
		["type"] = 0,
		["exp"] = 8900000000000,
		["ability"] = 281,
		["lv"] = 263,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[264] = {
		["type"] = 0,
		["exp"] = 9800000000000,
		["ability"] = 282,
		["lv"] = 264,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[265] = {
		["type"] = 0,
		["exp"] = 17000000000000,
		["ability"] = 283,
		["lv"] = 265,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[266] = {
		["type"] = 0,
		["exp"] = 12000000000000,
		["ability"] = 284,
		["lv"] = 266,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[267] = {
		["type"] = 0,
		["exp"] = 13000000000000,
		["ability"] = 285,
		["lv"] = 267,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
	[268] = {
		["type"] = 0,
		["exp"] = 14000000000000,
		["ability"] = 286,
		["lv"] = 268,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 300,
	},
}