return {
	[1] = {
		["type"] = 1,
		["exp"] = 5,
		["ability"] = 16,
		["lv"] = 1,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 16,
	},
	[2] = {
		["type"] = 1,
		["exp"] = 10,
		["ability"] = 17,
		["lv"] = 2,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 17,
	},
	[3] = {
		["type"] = 1,
		["exp"] = 25,
		["ability"] = 18,
		["lv"] = 3,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 18,
	},
	[4] = {
		["type"] = 1,
		["exp"] = 60,
		["ability"] = 19,
		["lv"] = 4,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 19,
	},
	[5] = {
		["type"] = 1,
		["exp"] = 100,
		["ability"] = 20,
		["lv"] = 5,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 20,
	},
	[6] = {
		["type"] = 1,
		["exp"] = 120,
		["ability"] = 21,
		["lv"] = 6,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 21,
	},
	[7] = {
		["type"] = 1,
		["exp"] = 140,
		["ability"] = 22,
		["lv"] = 7,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 22,
	},
	[8] = {
		["type"] = 1,
		["exp"] = 170,
		["ability"] = 23,
		["lv"] = 8,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 23,
	},
	[9] = {
		["type"] = 1,
		["exp"] = 210,
		["ability"] = 24,
		["lv"] = 9,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 24,
	},
	[10] = {
		["type"] = 1,
		["exp"] = 250,
		["ability"] = 25,
		["lv"] = 10,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 25,
	},
	[11] = {
		["type"] = 1,
		["exp"] = 300,
		["ability"] = 26,
		["lv"] = 11,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 26,
	},
	[12] = {
		["type"] = 1,
		["exp"] = 360,
		["ability"] = 27,
		["lv"] = 12,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 27,
	},
	[13] = {
		["type"] = 1,
		["exp"] = 430,
		["ability"] = 28,
		["lv"] = 13,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 28,
	},
	[14] = {
		["type"] = 1,
		["exp"] = 520,
		["ability"] = 29,
		["lv"] = 14,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 29,
	},
	[15] = {
		["type"] = 1,
		["exp"] = 620,
		["ability"] = 30,
		["lv"] = 15,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 30,
	},
	[16] = {
		["type"] = 1,
		["exp"] = 740,
		["ability"] = 31,
		["lv"] = 16,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 31,
	},
	[17] = {
		["type"] = 1,
		["exp"] = 890,
		["ability"] = 32,
		["lv"] = 17,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 32,
	},
	[18] = {
		["type"] = 1,
		["exp"] = 1100,
		["ability"] = 33,
		["lv"] = 18,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 33,
	},
	[19] = {
		["type"] = 1,
		["exp"] = 1300,
		["ability"] = 34,
		["lv"] = 19,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 34,
	},
	[20] = {
		["type"] = 1,
		["exp"] = 1500,
		["ability"] = 35,
		["lv"] = 20,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 35,
	},
	[21] = {
		["type"] = 1,
		["exp"] = 1700,
		["ability"] = 36,
		["lv"] = 21,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 36,
	},
	[22] = {
		["type"] = 1,
		["exp"] = 1900,
		["ability"] = 37,
		["lv"] = 22,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 37,
	},
	[23] = {
		["type"] = 1,
		["exp"] = 2100,
		["ability"] = 38,
		["lv"] = 23,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 38,
	},
	[24] = {
		["type"] = 1,
		["exp"] = 2300,
		["ability"] = 39,
		["lv"] = 24,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 39,
	},
	[25] = {
		["type"] = 1,
		["exp"] = 2500,
		["ability"] = 40,
		["lv"] = 25,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 40,
	},
	[26] = {
		["type"] = 1,
		["exp"] = 2700,
		["ability"] = 41,
		["lv"] = 26,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 41,
	},
	[27] = {
		["type"] = 1,
		["exp"] = 3000,
		["ability"] = 42,
		["lv"] = 27,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 42,
	},
	[28] = {
		["type"] = 1,
		["exp"] = 3300,
		["ability"] = 43,
		["lv"] = 28,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 43,
	},
	[29] = {
		["type"] = 1,
		["exp"] = 3600,
		["ability"] = 44,
		["lv"] = 29,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 44,
	},
	[30] = {
		["type"] = 1,
		["exp"] = 4000,
		["ability"] = 45,
		["lv"] = 30,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 45,
	},
	[31] = {
		["type"] = 1,
		["exp"] = 4400,
		["ability"] = 46,
		["lv"] = 31,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 46,
	},
	[32] = {
		["type"] = 1,
		["exp"] = 4800,
		["ability"] = 47,
		["lv"] = 32,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 47,
	},
	[33] = {
		["type"] = 1,
		["exp"] = 5300,
		["ability"] = 48,
		["lv"] = 33,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 48,
	},
	[34] = {
		["type"] = 1,
		["exp"] = 5900,
		["ability"] = 49,
		["lv"] = 34,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 49,
	},
	[35] = {
		["type"] = 1,
		["exp"] = 6400,
		["ability"] = 50,
		["lv"] = 35,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 50,
	},
	[36] = {
		["type"] = 1,
		["exp"] = 7100,
		["ability"] = 51,
		["lv"] = 36,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 51,
	},
	[37] = {
		["type"] = 1,
		["exp"] = 7800,
		["ability"] = 52,
		["lv"] = 37,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 52,
	},
	[38] = {
		["type"] = 1,
		["exp"] = 8600,
		["ability"] = 53,
		["lv"] = 38,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 53,
	},
	[39] = {
		["type"] = 1,
		["exp"] = 9400,
		["ability"] = 54,
		["lv"] = 39,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 54,
	},
	[40] = {
		["type"] = 1,
		["exp"] = 10000,
		["ability"] = 55,
		["lv"] = 40,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 55,
	},
	[41] = {
		["type"] = 1,
		["exp"] = 11000,
		["ability"] = 56,
		["lv"] = 41,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 56,
	},
	[42] = {
		["type"] = 1,
		["exp"] = 13000,
		["ability"] = 57,
		["lv"] = 42,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 57,
	},
	[43] = {
		["type"] = 1,
		["exp"] = 14000,
		["ability"] = 58,
		["lv"] = 43,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 58,
	},
	[44] = {
		["type"] = 1,
		["exp"] = 15000,
		["ability"] = 59,
		["lv"] = 44,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 59,
	},
	[45] = {
		["type"] = 1,
		["exp"] = 17000,
		["ability"] = 60,
		["lv"] = 45,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 60,
	},
	[46] = {
		["type"] = 1,
		["exp"] = 18000,
		["ability"] = 61,
		["lv"] = 46,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 61,
	},
	[47] = {
		["type"] = 1,
		["exp"] = 20000,
		["ability"] = 62,
		["lv"] = 47,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 62,
	},
	[48] = {
		["type"] = 1,
		["exp"] = 22000,
		["ability"] = 63,
		["lv"] = 48,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 63,
	},
	[49] = {
		["type"] = 1,
		["exp"] = 24000,
		["ability"] = 64,
		["lv"] = 49,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 64,
	},
	[50] = {
		["type"] = 1,
		["exp"] = 27000,
		["ability"] = 65,
		["lv"] = 50,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 65,
	},
	[51] = {
		["type"] = 1,
		["exp"] = 30000,
		["ability"] = 66,
		["lv"] = 51,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 66,
	},
	[52] = {
		["type"] = 1,
		["exp"] = 33000,
		["ability"] = 67,
		["lv"] = 52,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 67,
	},
	[53] = {
		["type"] = 1,
		["exp"] = 36000,
		["ability"] = 68,
		["lv"] = 53,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 68,
	},
	[54] = {
		["type"] = 1,
		["exp"] = 39000,
		["ability"] = 69,
		["lv"] = 54,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 69,
	},
	[55] = {
		["type"] = 1,
		["exp"] = 43000,
		["ability"] = 70,
		["lv"] = 55,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 70,
	},
	[56] = {
		["type"] = 1,
		["exp"] = 48000,
		["ability"] = 71,
		["lv"] = 56,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 71,
	},
	[57] = {
		["type"] = 1,
		["exp"] = 52000,
		["ability"] = 72,
		["lv"] = 57,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 72,
	},
	[58] = {
		["type"] = 1,
		["exp"] = 58000,
		["ability"] = 73,
		["lv"] = 58,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 73,
	},
	[59] = {
		["type"] = 1,
		["exp"] = 63000,
		["ability"] = 74,
		["lv"] = 59,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 74,
	},
	[60] = {
		["type"] = 1,
		["exp"] = 70000,
		["ability"] = 75,
		["lv"] = 60,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 75,
	},
	[61] = {
		["type"] = 1,
		["exp"] = 77000,
		["ability"] = 76,
		["lv"] = 61,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 76,
	},
	[62] = {
		["type"] = 1,
		["exp"] = 84000,
		["ability"] = 77,
		["lv"] = 62,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 77,
	},
	[63] = {
		["type"] = 1,
		["exp"] = 93000,
		["ability"] = 78,
		["lv"] = 63,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 78,
	},
	[64] = {
		["type"] = 1,
		["exp"] = 100000,
		["ability"] = 79,
		["lv"] = 64,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 79,
	},
	[65] = {
		["type"] = 1,
		["exp"] = 110000,
		["ability"] = 80,
		["lv"] = 65,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 80,
	},
	[66] = {
		["type"] = 1,
		["exp"] = 120000,
		["ability"] = 81,
		["lv"] = 66,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 81,
	},
	[67] = {
		["type"] = 1,
		["exp"] = 140000,
		["ability"] = 82,
		["lv"] = 67,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 82,
	},
	[68] = {
		["type"] = 1,
		["exp"] = 150000,
		["ability"] = 83,
		["lv"] = 68,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 83,
	},
	[69] = {
		["type"] = 1,
		["exp"] = 160000,
		["ability"] = 84,
		["lv"] = 69,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 84,
	},
	[70] = {
		["type"] = 1,
		["exp"] = 180000,
		["ability"] = 85,
		["lv"] = 70,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 85,
	},
	[71] = {
		["type"] = 1,
		["exp"] = 200000,
		["ability"] = 86,
		["lv"] = 71,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 86,
	},
	[72] = {
		["type"] = 1,
		["exp"] = 220000,
		["ability"] = 87,
		["lv"] = 72,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 87,
	},
	[73] = {
		["type"] = 1,
		["exp"] = 240000,
		["ability"] = 88,
		["lv"] = 73,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 88,
	},
	[74] = {
		["type"] = 1,
		["exp"] = 260000,
		["ability"] = 89,
		["lv"] = 74,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 89,
	},
	[75] = {
		["type"] = 1,
		["exp"] = 290000,
		["ability"] = 90,
		["lv"] = 75,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 90,
	},
	[76] = {
		["type"] = 1,
		["exp"] = 320000,
		["ability"] = 91,
		["lv"] = 76,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 91,
	},
	[77] = {
		["type"] = 1,
		["exp"] = 350000,
		["ability"] = 92,
		["lv"] = 77,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 92,
	},
	[78] = {
		["type"] = 1,
		["exp"] = 390000,
		["ability"] = 93,
		["lv"] = 78,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 93,
	},
	[79] = {
		["type"] = 1,
		["exp"] = 430000,
		["ability"] = 94,
		["lv"] = 79,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 94,
	},
	[80] = {
		["type"] = 1,
		["exp"] = 470000,
		["ability"] = 95,
		["lv"] = 80,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 95,
	},
	[81] = {
		["type"] = 1,
		["exp"] = 520000,
		["ability"] = 96,
		["lv"] = 81,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 96,
	},
	[82] = {
		["type"] = 1,
		["exp"] = 570000,
		["ability"] = 97,
		["lv"] = 82,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 97,
	},
	[83] = {
		["type"] = 1,
		["exp"] = 620000,
		["ability"] = 98,
		["lv"] = 83,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 98,
	},
	[84] = {
		["type"] = 1,
		["exp"] = 690000,
		["ability"] = 99,
		["lv"] = 84,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 99,
	},
	[85] = {
		["type"] = 1,
		["exp"] = 760000,
		["ability"] = 100,
		["lv"] = 85,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 100,
	},
	[86] = {
		["type"] = 1,
		["exp"] = 830000,
		["ability"] = 101,
		["lv"] = 86,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 100.5,
	},
	[87] = {
		["type"] = 1,
		["exp"] = 910000,
		["ability"] = 102,
		["lv"] = 87,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 101,
	},
	[88] = {
		["type"] = 1,
		["exp"] = 1000000,
		["ability"] = 103,
		["lv"] = 88,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 101.5,
	},
	[89] = {
		["type"] = 1,
		["exp"] = 1100000,
		["ability"] = 104,
		["lv"] = 89,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 102,
	},
	[90] = {
		["type"] = 1,
		["exp"] = 1200000,
		["ability"] = 105,
		["lv"] = 90,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 102.5,
	},
	[91] = {
		["type"] = 1,
		["exp"] = 1300000,
		["ability"] = 106,
		["lv"] = 91,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 103,
	},
	[92] = {
		["type"] = 1,
		["exp"] = 1500000,
		["ability"] = 107,
		["lv"] = 92,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 103.5,
	},
	[93] = {
		["type"] = 1,
		["exp"] = 1600000,
		["ability"] = 108,
		["lv"] = 93,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 104,
	},
	[94] = {
		["type"] = 1,
		["exp"] = 1800000,
		["ability"] = 109,
		["lv"] = 94,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 104.5,
	},
	[95] = {
		["type"] = 1,
		["exp"] = 2000000,
		["ability"] = 110,
		["lv"] = 95,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 105,
	},
	[96] = {
		["type"] = 1,
		["exp"] = 2200000,
		["ability"] = 111,
		["lv"] = 96,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 105.5,
	},
	[97] = {
		["type"] = 1,
		["exp"] = 2400000,
		["ability"] = 112,
		["lv"] = 97,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 106,
	},
	[98] = {
		["type"] = 1,
		["exp"] = 2600000,
		["ability"] = 113,
		["lv"] = 98,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 106.5,
	},
	[99] = {
		["type"] = 1,
		["exp"] = 2900000,
		["ability"] = 114,
		["lv"] = 99,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 107,
	},
	[100] = {
		["type"] = 1,
		["exp"] = 3200000,
		["ability"] = 115,
		["lv"] = 100,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 107.5,
	},
	[101] = {
		["type"] = 1,
		["exp"] = 3500000,
		["ability"] = 116,
		["lv"] = 101,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 108,
	},
	[102] = {
		["type"] = 1,
		["exp"] = 3800000,
		["ability"] = 117,
		["lv"] = 102,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 108.5,
	},
	[103] = {
		["type"] = 1,
		["exp"] = 4200000,
		["ability"] = 118,
		["lv"] = 103,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 109,
	},
	[104] = {
		["type"] = 1,
		["exp"] = 4600000,
		["ability"] = 119,
		["lv"] = 104,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 109.5,
	},
	[105] = {
		["type"] = 1,
		["exp"] = 5100000,
		["ability"] = 120,
		["lv"] = 105,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 110,
	},
	[106] = {
		["type"] = 1,
		["exp"] = 5600000,
		["ability"] = 121,
		["lv"] = 106,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 110.5,
	},
	[107] = {
		["type"] = 1,
		["exp"] = 6200000,
		["ability"] = 122,
		["lv"] = 107,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 111,
	},
	[108] = {
		["type"] = 1,
		["exp"] = 6800000,
		["ability"] = 123,
		["lv"] = 108,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 111.5,
	},
	[109] = {
		["type"] = 1,
		["exp"] = 7400000,
		["ability"] = 124,
		["lv"] = 109,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 112,
	},
	[110] = {
		["type"] = 1,
		["exp"] = 8200000,
		["ability"] = 125,
		["lv"] = 110,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 112.5,
	},
	[111] = {
		["type"] = 1,
		["exp"] = 9000000,
		["ability"] = 126,
		["lv"] = 111,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 113,
	},
	[112] = {
		["type"] = 1,
		["exp"] = 9900000,
		["ability"] = 127,
		["lv"] = 112,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 113.5,
	},
	[113] = {
		["type"] = 1,
		["exp"] = 11000000,
		["ability"] = 128,
		["lv"] = 113,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 114,
	},
	[114] = {
		["type"] = 1,
		["exp"] = 12000000,
		["ability"] = 129,
		["lv"] = 114,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 114.5,
	},
	[115] = {
		["type"] = 1,
		["exp"] = 13000000,
		["ability"] = 130,
		["lv"] = 115,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 115,
	},
	[116] = {
		["type"] = 1,
		["exp"] = 15000000,
		["ability"] = 131,
		["lv"] = 116,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 115.5,
	},
	[117] = {
		["type"] = 1,
		["exp"] = 16000000,
		["ability"] = 132,
		["lv"] = 117,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 116,
	},
	[118] = {
		["type"] = 1,
		["exp"] = 18000000,
		["ability"] = 133,
		["lv"] = 118,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 116.5,
	},
	[119] = {
		["type"] = 1,
		["exp"] = 19000000,
		["ability"] = 134,
		["lv"] = 119,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 117,
	},
	[120] = {
		["type"] = 1,
		["exp"] = 21000000,
		["ability"] = 135,
		["lv"] = 120,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 117.5,
	},
	[121] = {
		["type"] = 1,
		["exp"] = 23000000,
		["ability"] = 136,
		["lv"] = 121,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 118,
	},
	[122] = {
		["type"] = 1,
		["exp"] = 26000000,
		["ability"] = 137,
		["lv"] = 122,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 118.5,
	},
	[123] = {
		["type"] = 1,
		["exp"] = 28000000,
		["ability"] = 138,
		["lv"] = 123,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 119,
	},
	[124] = {
		["type"] = 1,
		["exp"] = 31000000,
		["ability"] = 139,
		["lv"] = 124,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 119.5,
	},
	[125] = {
		["type"] = 1,
		["exp"] = 34000000,
		["ability"] = 140,
		["lv"] = 125,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 120,
	},
	[126] = {
		["type"] = 1,
		["exp"] = 38000000,
		["ability"] = 141,
		["lv"] = 126,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 120.5,
	},
	[127] = {
		["type"] = 1,
		["exp"] = 41000000,
		["ability"] = 142,
		["lv"] = 127,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 121,
	},
	[128] = {
		["type"] = 1,
		["exp"] = 46000000,
		["ability"] = 143,
		["lv"] = 128,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 121.5,
	},
	[129] = {
		["type"] = 1,
		["exp"] = 50000000,
		["ability"] = 144,
		["lv"] = 129,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 122,
	},
	[130] = {
		["type"] = 1,
		["exp"] = 55000000,
		["ability"] = 145,
		["lv"] = 130,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 122.5,
	},
	[131] = {
		["type"] = 1,
		["exp"] = 61000000,
		["ability"] = 146,
		["lv"] = 131,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 123,
	},
	[132] = {
		["type"] = 1,
		["exp"] = 67000000,
		["ability"] = 147,
		["lv"] = 132,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 123.5,
	},
	[133] = {
		["type"] = 1,
		["exp"] = 73000000,
		["ability"] = 148,
		["lv"] = 133,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 124,
	},
	[134] = {
		["type"] = 1,
		["exp"] = 81000000,
		["ability"] = 149,
		["lv"] = 134,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 124.5,
	},
	[135] = {
		["type"] = 1,
		["exp"] = 89000000,
		["ability"] = 150,
		["lv"] = 135,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 125,
	},
	[136] = {
		["type"] = 0,
		["exp"] = 98000000,
		["ability"] = 151,
		["lv"] = 136,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 125.5,
	},
	[137] = {
		["type"] = 0,
		["exp"] = 110000000,
		["ability"] = 152,
		["lv"] = 137,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 126,
	},
	[138] = {
		["type"] = 0,
		["exp"] = 120000000,
		["ability"] = 153,
		["lv"] = 138,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 126.5,
	},
	[139] = {
		["type"] = 0,
		["exp"] = 130000000,
		["ability"] = 154,
		["lv"] = 139,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 127,
	},
	[140] = {
		["type"] = 0,
		["exp"] = 140000000,
		["ability"] = 155,
		["lv"] = 140,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 127.5,
	},
	[141] = {
		["type"] = 0,
		["exp"] = 160000000,
		["ability"] = 156,
		["lv"] = 141,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 128,
	},
	[142] = {
		["type"] = 0,
		["exp"] = 170000000,
		["ability"] = 157,
		["lv"] = 142,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 128.5,
	},
	[143] = {
		["type"] = 0,
		["exp"] = 190000000,
		["ability"] = 158,
		["lv"] = 143,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 129,
	},
	[144] = {
		["type"] = 0,
		["exp"] = 210000000,
		["ability"] = 159,
		["lv"] = 144,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 129.5,
	},
	[145] = {
		["type"] = 0,
		["exp"] = 230000000,
		["ability"] = 160,
		["lv"] = 145,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 130,
	},
	[146] = {
		["type"] = 0,
		["exp"] = 250000000,
		["ability"] = 161,
		["lv"] = 146,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 130.5,
	},
	[147] = {
		["type"] = 0,
		["exp"] = 280000000,
		["ability"] = 162,
		["lv"] = 147,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 131,
	},
	[148] = {
		["type"] = 0,
		["exp"] = 310000000,
		["ability"] = 163,
		["lv"] = 148,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 131.5,
	},
	[149] = {
		["type"] = 0,
		["exp"] = 340000000,
		["ability"] = 164,
		["lv"] = 149,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 132,
	},
	[150] = {
		["type"] = 0,
		["exp"] = 370000000,
		["ability"] = 165,
		["lv"] = 150,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 132.5,
	},
	[151] = {
		["type"] = 0,
		["exp"] = 410000000,
		["ability"] = 166,
		["lv"] = 151,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 133,
	},
	[152] = {
		["type"] = 0,
		["exp"] = 450000000,
		["ability"] = 167,
		["lv"] = 152,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 133.5,
	},
	[153] = {
		["type"] = 0,
		["exp"] = 490000000,
		["ability"] = 168,
		["lv"] = 153,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 134,
	},
	[154] = {
		["type"] = 0,
		["exp"] = 540000000,
		["ability"] = 169,
		["lv"] = 154,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 134.5,
	},
	[155] = {
		["type"] = 0,
		["exp"] = 600000000,
		["ability"] = 170,
		["lv"] = 155,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 135,
	},
	[156] = {
		["type"] = 0,
		["exp"] = 660000000,
		["ability"] = 171,
		["lv"] = 156,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 135.5,
	},
	[157] = {
		["type"] = 0,
		["exp"] = 720000000,
		["ability"] = 172,
		["lv"] = 157,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 136,
	},
	[158] = {
		["type"] = 0,
		["exp"] = 790000000,
		["ability"] = 173,
		["lv"] = 158,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 136.5,
	},
	[159] = {
		["type"] = 0,
		["exp"] = 870000000,
		["ability"] = 174,
		["lv"] = 159,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 137,
	},
	[160] = {
		["type"] = 0,
		["exp"] = 960000000,
		["ability"] = 175,
		["lv"] = 160,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 137.5,
	},
	[161] = {
		["type"] = 0,
		["exp"] = 1100000000,
		["ability"] = 176,
		["lv"] = 161,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 138,
	},
	[162] = {
		["type"] = 0,
		["exp"] = 1200000000,
		["ability"] = 177,
		["lv"] = 162,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 138.5,
	},
	[163] = {
		["type"] = 0,
		["exp"] = 1300000000,
		["ability"] = 178,
		["lv"] = 163,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 139,
	},
	[164] = {
		["type"] = 0,
		["exp"] = 1400000000,
		["ability"] = 179,
		["lv"] = 164,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 139.5,
	},
	[165] = {
		["type"] = 0,
		["exp"] = 1500000000,
		["ability"] = 180,
		["lv"] = 165,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 140,
	},
	[166] = {
		["type"] = 0,
		["exp"] = 1700000000,
		["ability"] = 181,
		["lv"] = 166,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 140.5,
	},
	[167] = {
		["type"] = 0,
		["exp"] = 1900000000,
		["ability"] = 182,
		["lv"] = 167,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 141,
	},
	[168] = {
		["type"] = 0,
		["exp"] = 2100000000,
		["ability"] = 183,
		["lv"] = 168,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 141.5,
	},
	[169] = {
		["type"] = 0,
		["exp"] = 2300000000,
		["ability"] = 184,
		["lv"] = 169,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 142,
	},
	[170] = {
		["type"] = 0,
		["exp"] = 2500000000,
		["ability"] = 185,
		["lv"] = 170,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 142.5,
	},
	[171] = {
		["type"] = 0,
		["exp"] = 2700000000,
		["ability"] = 186,
		["lv"] = 171,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 143,
	},
	[172] = {
		["type"] = 0,
		["exp"] = 3000000000,
		["ability"] = 187,
		["lv"] = 172,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 143.5,
	},
	[173] = {
		["type"] = 0,
		["exp"] = 3300000000,
		["ability"] = 188,
		["lv"] = 173,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 144,
	},
	[174] = {
		["type"] = 0,
		["exp"] = 3600000000,
		["ability"] = 189,
		["lv"] = 174,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 144.5,
	},
	[175] = {
		["type"] = 0,
		["exp"] = 4000000000,
		["ability"] = 190,
		["lv"] = 175,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 145,
	},
	[176] = {
		["type"] = 0,
		["exp"] = 4400000000,
		["ability"] = 191,
		["lv"] = 176,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 145.5,
	},
	[177] = {
		["type"] = 0,
		["exp"] = 4900000000,
		["ability"] = 192,
		["lv"] = 177,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 146,
	},
	[178] = {
		["type"] = 0,
		["exp"] = 5300000000,
		["ability"] = 193,
		["lv"] = 178,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 146.5,
	},
	[179] = {
		["type"] = 0,
		["exp"] = 5900000000,
		["ability"] = 194,
		["lv"] = 179,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 147,
	},
	[180] = {
		["type"] = 0,
		["exp"] = 6500000000,
		["ability"] = 195,
		["lv"] = 180,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 147.5,
	},
	[181] = {
		["type"] = 0,
		["exp"] = 7100000000,
		["ability"] = 196,
		["lv"] = 181,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 148,
	},
	[182] = {
		["type"] = 0,
		["exp"] = 7800000000,
		["ability"] = 197,
		["lv"] = 182,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 148.5,
	},
	[183] = {
		["type"] = 0,
		["exp"] = 8600000000,
		["ability"] = 198,
		["lv"] = 183,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 149,
	},
	[184] = {
		["type"] = 0,
		["exp"] = 9500000000,
		["ability"] = 199,
		["lv"] = 184,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 149.5,
	},
	[185] = {
		["type"] = 0,
		["exp"] = 10000000000,
		["ability"] = 200,
		["lv"] = 185,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 150,
	},
	[186] = {
		["type"] = 0,
		["exp"] = 11000000000,
		["ability"] = 201,
		["lv"] = 186,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 150.5,
	},
	[187] = {
		["type"] = 0,
		["exp"] = 13000000000,
		["ability"] = 202,
		["lv"] = 187,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 151,
	},
	[188] = {
		["type"] = 0,
		["exp"] = 14000000000,
		["ability"] = 203,
		["lv"] = 188,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 151.5,
	},
	[189] = {
		["type"] = 0,
		["exp"] = 15000000000,
		["ability"] = 204,
		["lv"] = 189,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 152,
	},
	[190] = {
		["type"] = 0,
		["exp"] = 17000000000,
		["ability"] = 205,
		["lv"] = 190,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 152.5,
	},
	[191] = {
		["type"] = 0,
		["exp"] = 18000000000,
		["ability"] = 206,
		["lv"] = 191,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 153,
	},
	[192] = {
		["type"] = 0,
		["exp"] = 20000000000,
		["ability"] = 207,
		["lv"] = 192,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 153.5,
	},
	[193] = {
		["type"] = 0,
		["exp"] = 22000000000,
		["ability"] = 208,
		["lv"] = 193,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 154,
	},
	[194] = {
		["type"] = 0,
		["exp"] = 25000000000,
		["ability"] = 209,
		["lv"] = 194,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 154.5,
	},
	[195] = {
		["type"] = 0,
		["exp"] = 27000000000,
		["ability"] = 210,
		["lv"] = 195,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 155,
	},
	[196] = {
		["type"] = 0,
		["exp"] = 30000000000,
		["ability"] = 211,
		["lv"] = 196,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 155.5,
	},
	[197] = {
		["type"] = 0,
		["exp"] = 33000000000,
		["ability"] = 212,
		["lv"] = 197,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 156,
	},
	[198] = {
		["type"] = 0,
		["exp"] = 36000000000,
		["ability"] = 213,
		["lv"] = 198,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 156.5,
	},
	[199] = {
		["type"] = 0,
		["exp"] = 40000000000,
		["ability"] = 214,
		["lv"] = 199,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 157,
	},
	[200] = {
		["type"] = 0,
		["exp"] = 43000000000,
		["ability"] = 215,
		["lv"] = 200,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 157.5,
	},
	[201] = {
		["type"] = 0,
		["exp"] = 48000000000,
		["ability"] = 216,
		["lv"] = 201,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 158,
	},
	[202] = {
		["type"] = 0,
		["exp"] = 53000000000,
		["ability"] = 217,
		["lv"] = 202,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 158.5,
	},
	[203] = {
		["type"] = 0,
		["exp"] = 58000000000,
		["ability"] = 218,
		["lv"] = 203,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 159,
	},
	[204] = {
		["type"] = 0,
		["exp"] = 64000000000,
		["ability"] = 219,
		["lv"] = 204,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 159.5,
	},
	[205] = {
		["type"] = 0,
		["exp"] = 70000000000,
		["ability"] = 220,
		["lv"] = 205,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 160,
	},
	[206] = {
		["type"] = 0,
		["exp"] = 77000000000,
		["ability"] = 221,
		["lv"] = 206,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 160.5,
	},
	[207] = {
		["type"] = 0,
		["exp"] = 85000000000,
		["ability"] = 222,
		["lv"] = 207,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 161,
	},
	[208] = {
		["type"] = 0,
		["exp"] = 93000000000,
		["ability"] = 223,
		["lv"] = 208,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 161.5,
	},
	[209] = {
		["type"] = 0,
		["exp"] = 100000000000,
		["ability"] = 224,
		["lv"] = 209,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 162,
	},
	[210] = {
		["type"] = 0,
		["exp"] = 110000000000,
		["ability"] = 225,
		["lv"] = 210,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 162.5,
	},
	[211] = {
		["type"] = 0,
		["exp"] = 120000000000,
		["ability"] = 226,
		["lv"] = 211,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 163,
	},
	[212] = {
		["type"] = 0,
		["exp"] = 140000000000,
		["ability"] = 227,
		["lv"] = 212,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 163.5,
	},
	[213] = {
		["type"] = 0,
		["exp"] = 150000000000,
		["ability"] = 228,
		["lv"] = 213,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 164,
	},
	[214] = {
		["type"] = 0,
		["exp"] = 170000000000,
		["ability"] = 229,
		["lv"] = 214,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 164.5,
	},
	[215] = {
		["type"] = 0,
		["exp"] = 180000000000,
		["ability"] = 230,
		["lv"] = 215,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 165,
	},
	[216] = {
		["type"] = 0,
		["exp"] = 200000000000,
		["ability"] = 231,
		["lv"] = 216,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 165.5,
	},
	[217] = {
		["type"] = 0,
		["exp"] = 220000000000,
		["ability"] = 232,
		["lv"] = 217,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 166,
	},
	[218] = {
		["type"] = 0,
		["exp"] = 240000000000,
		["ability"] = 233,
		["lv"] = 218,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 166.5,
	},
	[219] = {
		["type"] = 0,
		["exp"] = 270000000000,
		["ability"] = 234,
		["lv"] = 219,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 167,
	},
	[220] = {
		["type"] = 0,
		["exp"] = 290000000000,
		["ability"] = 235,
		["lv"] = 220,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 167.5,
	},
	[221] = {
		["type"] = 0,
		["exp"] = 320000000000,
		["ability"] = 236,
		["lv"] = 221,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 168,
	},
	[222] = {
		["type"] = 0,
		["exp"] = 350000000000,
		["ability"] = 237,
		["lv"] = 222,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 168.5,
	},
	[223] = {
		["type"] = 0,
		["exp"] = 390000000000,
		["ability"] = 238,
		["lv"] = 223,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 169,
	},
	[224] = {
		["type"] = 0,
		["exp"] = 430000000000,
		["ability"] = 239,
		["lv"] = 224,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 169.5,
	},
	[225] = {
		["type"] = 0,
		["exp"] = 470000000000,
		["ability"] = 240,
		["lv"] = 225,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 170,
	},
	[226] = {
		["type"] = 0,
		["exp"] = 520000000000,
		["ability"] = 241,
		["lv"] = 226,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 170.5,
	},
	[227] = {
		["type"] = 0,
		["exp"] = 570000000000,
		["ability"] = 242,
		["lv"] = 227,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 171,
	},
	[228] = {
		["type"] = 0,
		["exp"] = 630000000000,
		["ability"] = 243,
		["lv"] = 228,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 171.5,
	},
	[229] = {
		["type"] = 0,
		["exp"] = 690000000000,
		["ability"] = 244,
		["lv"] = 229,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 172,
	},
	[230] = {
		["type"] = 0,
		["exp"] = 760000000000,
		["ability"] = 245,
		["lv"] = 230,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 172.5,
	},
	[231] = {
		["type"] = 0,
		["exp"] = 830000000000,
		["ability"] = 246,
		["lv"] = 231,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 173,
	},
	[232] = {
		["type"] = 0,
		["exp"] = 920000000000,
		["ability"] = 247,
		["lv"] = 232,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 173.5,
	},
	[233] = {
		["type"] = 0,
		["exp"] = 1000000000000,
		["ability"] = 248,
		["lv"] = 233,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 174,
	},
	[234] = {
		["type"] = 0,
		["exp"] = 1100000000000,
		["ability"] = 249,
		["lv"] = 234,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 174.5,
	},
	[235] = {
		["type"] = 0,
		["exp"] = 1200000000000,
		["ability"] = 250,
		["lv"] = 235,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 175,
	},
	[236] = {
		["type"] = 0,
		["exp"] = 1300000000000,
		["ability"] = 251,
		["lv"] = 236,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 175.5,
	},
	[237] = {
		["type"] = 0,
		["exp"] = 1500000000000,
		["ability"] = 252,
		["lv"] = 237,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 176,
	},
	[238] = {
		["type"] = 0,
		["exp"] = 1600000000000,
		["ability"] = 253,
		["lv"] = 238,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 176.5,
	},
	[239] = {
		["type"] = 0,
		["exp"] = 1800000000000,
		["ability"] = 254,
		["lv"] = 239,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 177,
	},
	[240] = {
		["type"] = 0,
		["exp"] = 2000000000000,
		["ability"] = 255,
		["lv"] = 240,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 177.5,
	},
	[241] = {
		["type"] = 0,
		["exp"] = 2200000000000,
		["ability"] = 256,
		["lv"] = 241,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 178,
	},
	[242] = {
		["type"] = 0,
		["exp"] = 2400000000000,
		["ability"] = 257,
		["lv"] = 242,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 178.5,
	},
	[243] = {
		["type"] = 0,
		["exp"] = 2600000000000,
		["ability"] = 258,
		["lv"] = 243,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 179,
	},
	[244] = {
		["type"] = 0,
		["exp"] = 2900000000000,
		["ability"] = 259,
		["lv"] = 244,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 179.5,
	},
	[245] = {
		["type"] = 0,
		["exp"] = 3200000000000,
		["ability"] = 260,
		["lv"] = 245,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 180,
	},
	[246] = {
		["type"] = 0,
		["exp"] = 3500000000000,
		["ability"] = 261,
		["lv"] = 246,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 180.5,
	},
	[247] = {
		["type"] = 0,
		["exp"] = 3800000000000,
		["ability"] = 262,
		["lv"] = 247,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 181,
	},
	[248] = {
		["type"] = 0,
		["exp"] = 4200000000000,
		["ability"] = 263,
		["lv"] = 248,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 181.5,
	},
	[249] = {
		["type"] = 0,
		["exp"] = 4600000000000,
		["ability"] = 264,
		["lv"] = 249,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 182,
	},
	[250] = {
		["type"] = 0,
		["exp"] = 5100000000000,
		["ability"] = 265,
		["lv"] = 250,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 182.5,
	},
	[251] = {
		["type"] = 0,
		["exp"] = 5600000000000,
		["ability"] = 266,
		["lv"] = 251,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 183,
	},
	[252] = {
		["type"] = 0,
		["exp"] = 6200000000000,
		["ability"] = 267,
		["lv"] = 252,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 183.5,
	},
	[253] = {
		["type"] = 0,
		["exp"] = 6800000000000,
		["ability"] = 268,
		["lv"] = 253,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 184,
	},
	[254] = {
		["type"] = 0,
		["exp"] = 7500000000000,
		["ability"] = 269,
		["lv"] = 254,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 184.5,
	},
	[255] = {
		["type"] = 0,
		["exp"] = 8200000000000,
		["ability"] = 270,
		["lv"] = 255,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 185,
	},
	[256] = {
		["type"] = 0,
		["exp"] = 9000000000000,
		["ability"] = 271,
		["lv"] = 256,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 185.5,
	},
	[257] = {
		["type"] = 0,
		["exp"] = 9900000000000,
		["ability"] = 272,
		["lv"] = 257,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 186,
	},
	[258] = {
		["type"] = 0,
		["exp"] = 11000000000000,
		["ability"] = 273,
		["lv"] = 258,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 186.5,
	},
	[259] = {
		["type"] = 0,
		["exp"] = 12000000000000,
		["ability"] = 274,
		["lv"] = 259,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 187,
	},
	[260] = {
		["type"] = 0,
		["exp"] = 13000000000000,
		["ability"] = 275,
		["lv"] = 260,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 187.5,
	},
	[261] = {
		["type"] = 0,
		["exp"] = 15000000000000,
		["ability"] = 276,
		["lv"] = 261,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 188,
	},
	[262] = {
		["type"] = 0,
		["exp"] = 16000000000000,
		["ability"] = 277,
		["lv"] = 262,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 188.5,
	},
	[263] = {
		["type"] = 0,
		["exp"] = 18000000000000,
		["ability"] = 278,
		["lv"] = 263,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 189,
	},
	[264] = {
		["type"] = 0,
		["exp"] = 19000000000000,
		["ability"] = 279,
		["lv"] = 264,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 189.5,
	},
	[265] = {
		["type"] = 0,
		["exp"] = 21000000000000,
		["ability"] = 280,
		["lv"] = 265,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 190,
	},
	[266] = {
		["type"] = 0,
		["exp"] = 23000000000000,
		["ability"] = 281,
		["lv"] = 266,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 190.5,
	},
	[267] = {
		["type"] = 0,
		["exp"] = 26000000000000,
		["ability"] = 282,
		["lv"] = 267,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 191,
	},
	[268] = {
		["type"] = 0,
		["exp"] = 28000000000000,
		["ability"] = 283,
		["lv"] = 268,
		["name_t"] = "Jumppower",
		["walkSpeed"] = 191.5,
	},
}