return {
	[1] = {
		["bgIconId"] = "rbxassetid://77886307507247",
		["type"] = 1,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 1,
		["price1"] = 19,
		["name_t"] = "Starter Pack",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 1,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602509411,
	},
	[2] = {
		["bgIconId"] = "rbxassetid://83983663285653",
		["type"] = 1,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 2,
		["price1"] = 49,
		["name_t"] = "Pro Pack",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 2,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602509374,
	},
	[3] = {
		["bgIconId"] = "rbxassetid://140006926894911",
		["type"] = 4,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 3,
		["price1"] = 268,
		["name_t"] = "X2 Exercise Efficiency",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 3,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602750616,
	},
	[4] = {
		["bgIconId"] = "rbxassetid://131609692287978",
		["type"] = 4,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 4,
		["price1"] = 298,
		["name_t"] = "X2 Coin",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 4,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602750670,
	},
}