return {
	[1] = {
		["bgIconId"] = "1",
		["type"] = 1,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 1,
		["price1"] = 19,
		["name_t"] = "Starter Pack",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 2,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602583418,
	},
	[2] = {
		["bgIconId"] = "2",
		["type"] = 1,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 2,
		["price1"] = 49,
		["name_t"] = "Pro Pack",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 3,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602583447,
	},
	[3] = {
		["bgIconId"] = "3",
		["type"] = 4,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 3,
		["price1"] = 268,
		["name_t"] = "X2 Exercise Efficiency",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 4,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602750616,
	},
	[4] = {
		["bgIconId"] = "4",
		["type"] = 4,
		["productId2"] = 0,
		["beforeDiscountPrice2"] = 0,
		["id"] = 4,
		["price1"] = 298,
		["name_t"] = "X2 Coin",
		["nameColor"] = {
			[1] = 255,
			[2] = 255,
			[3] = 255,
		},
		["price2"] = 0,
		["price3"] = 0,
		["beforeDiscountPrice3"] = 0,
		["canRepurchased"] = 0,
		["productId3"] = 0,
		["sortId"] = 5,
		["arePaidRandomItemsRestricted"] = 0,
		["productId1"] = 3602750670,
	},
	[5] = {
		["bgIconId"] = "5",
		["type"] = 2,
		["productId2"] = 3603365249,
		["beforeDiscountPrice2"] = 297,
		["id"] = 5,
		["price1"] = 99,
		["name_t"] = "Loop Chest",
		["nameColor"] = {
			[1] = 85,
			[2] = 255,
			[3] = 127,
		},
		["price2"] = 249,
		["price3"] = 888,
		["beforeDiscountPrice3"] = 990,
		["canRepurchased"] = 1,
		["productId3"] = 3603365278,
		["sortId"] = 1,
		["arePaidRandomItemsRestricted"] = 1,
		["productId1"] = 3603365225,
	},
}